---

title: Transgenic pig model for a hereditary neurodegenerative autosomal dominant disease
abstract: The present invention discloses a non-human animal model for a hereditary autosomal dominant disease. The non-human animal model expresses at least one phenotype associated with the disease and is obtained by a genetic determinant. The invention also relates to sperm cells and embryos comprising the genetic determinant for an autosomal dominant disease. Furthermore, methods for producing the non-human animal model, sperm cell, and embryos are disclosed.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08173861&OS=08173861&RS=08173861
owner: Aarhus Universitet
number: 08173861
owner_city: Aarhus C
owner_country: DK
publication_date: 20070430
---
This application is a 371 national phase filing of PCT DK2007 000204 filed Apr. 30 2007 and claims priority to Danish Application No. PA 2006 00616 filed May 1 2006 U.S. Provisional Application No. 60 796 196 filed May 1 2006 and Danish Application No. PA 2007 00429 filed Mar. 20 2007.

All patent and non patent references cited in the application are hereby incorporated by reference in their entirety.

The present invention relates to a non human animal model such as a porcine animal model and methods for producing a non human animal model by sperm mediated gene transfer SMGT . When a genetic determinant involved in a hereditary disease is used for SMGT i.e. a genetic determinant which confers a dominant phenotype for said hereditary disease the non human animal model can be used for studying such hereditary diseases such as autosomal dominant hereditary diseases for example protein conformation diseases such as Amyotrophic Lateral Sclerosis ALS Alzheimer s disease Parkinson s disease diseases related to trinucleotide repeats Huntington s disease but also dyschondroplasia.

Transgenic non human animals can be used to understand the action of a single gene in the context of the whole animal and the interrelated phenomena of gene activation expression and interaction. The technology has also led to the production of models for various diseases in humans and other animals which contributes significantly to an increased understanding of genetic mechanisms and of genes associated with specific diseases.

While smaller animals such as mice have proved to be suitable models for certain diseases their value as animal models for many human diseases is quite limited. Larger transgenic animals are much more suitable than mice for the study of many of the effects and treatments of most human diseases because of their greater similarity to humans in many aspects.

For the past two decades pigs have been used in biomedical research with increasing frequency as replacements for dog and primates. This is due to the anatomical and physiological similarity to humans. Pigs and human share anatomical and physiological characteristics such as heart size cardiac output and coronary blood supply which have made pigs widely used in cardiac surgery pacemaker studies and heart transplantions. Similarly pig and humans share features in digestive physiology and pigs are therefore widely used in nutritional studies and subjects in relations to this including lipid metabolism gastric ulceration diabetes and alcoholism Furthermore porcine models are used for the study of disorders of the skin. Organs of porcine origin are also used in organ transplantation research. However the pig constitutes an evolutionary clade in relation to humans and rodents.

Many human diseases are hereditary. The inheritance of genetic disorders abnormalities or traits is a function of both the type of chromosome on which the abnormal gene resides autosomal or sex chromosome and of the trait itself i.e. whether the trait is dominant or recessive. The trait can be due to a single defective gene from one parent dominant inheritance or the trait can arise when two copies of the gene one from each parent are defective recessive inheritance .

Dominant inheritance occurs when an abnormal gene from one parent is capable of causing disease even though the matching gene from the other parent is normal. Accordingly the abnormal gene dominates the outcome of the gene pair and one copy of the mutant gene is sufficient for expression of the abnormal phenotype.

Several distinct characteristics of autosomal dominant inheritance include Every affected individual has an affected parent except in cases of new mutations or incomplete penetrance males and females are equally likely to inherit the allele and be affected as the genes are located on autosomes of which each male and female has two copies and recurrence risk the probability that a genetic disorder that is present in a patient will recur in another member of the family for each child of an affected parent is as only one copy of a gene is necessary for development of the disease . If one parent is a heterozygote for a particular gene their offspring will either inherit the gene or they will not with each outcome equally likely. Accordingly if an affected individual s siblings are not affected they do not carry the mutation and cannot therefore pass it on to their own offspring.

As many of these autosomal dominant diseases are deleterious one would expect that over time they would disappear from the population due to natural selection. However there are several phenomena cf. below that can lead to maintenance of these alleles in the population.

Variable expressivity the variable severity of a genetic trait. Different individuals with the same mutation will develop different degrees of the disorder due to difference in environment and the modifying effects of other genes. Because of this a mutation that leads to a relatively mild form of the disease in one individual stands a good chance of being passed on and maintained in the population. The same mutation in another individual may lead to such a severe manifestation that the affected individual is unable to propagate the mutation to the next generation. This demonstrates very well the fact that genetic disease results as combination of genetic and environmental influences.

Late onset when a disease has an onset later in life affected individuals may have passed the gene to their offspring before they even knew they carried it themselves. One example of this is Huntington s disease a late onset neurodegenerative disorder. It is now possible to receive genetic testing for this disorder a practice that leads to many complex issues for the family undergoing the testing.

High recurrent mutation rate 85 of cases of achondroplasia a major cause of dwarfism are the result of new mutations. Some segments of the genome are subject to higher than normal rates of mutation which can lead to the maintenance of the disease in the population even if both parents were normal. This is particularly true of diseases that affect fertility. If the disease is invariably lethal at a young age before reproduction is possible the only source of the disease would be new mutations.

Incomplete penetrance phenomena where a portion of individuals with a disease associated genotype do not develop a disease. If only 30 people out of 50 who have a disease associated mutation actually develop the disease it is incompletely penetrant. A disease that is 75 penetrant is one in which 75 of those who carry the disease associated mutation eventually develop the disease. The rest do not.

Transgenic animals carrying a dominant disease gene which is expressed in the animal makes it possible to study the phenotype associated with said dominant disease gene if the gene when expressed in the animal actually leads to the same disease as in humans. Transgenic animals have traditionally been used for the improvement of livestock and for the large scale production of biologically active pharmaceuticals. Historically transgenic animals have been produced almost exclusively by microinjection of the fertilized egg. The pronuclei of fertilized eggs are microinjected in vitro with foreign i.e. xenogeneic or allogeneic DNA or hybrid DNA molecules. The microinjected fertilized eggs can then be transferred to the genital tract of a pseudopregnant female.

Only a few examples of success with sperm mediated gene transfer methods in monkeys and mice have been reported reviewed e.g. by Vodicka 2005 Ann. N.Y. Acad. Sci. 1049 161 171 Chan 2004 Reprod. Biol. Endocrinol. 2 39 and by Wall 2002 Theriogenology 57 189 201 .

As noted by Wall ibid only few studies convincingly demonstrate transgene expression. Wall ibid concludes that the body of evidence is still not sufficient to warrant elevating sperm mediated gene transfer to the status of other available state of the art methods.

Smith 2004 Int. J. Med. Sci. 1 2 76 91 notes that sperm mediated gene transfer has not yet become established as a reliable form of genetic modification and that concerted attempts to utilise sperm mediated gene transfer often have produced negative results.

JP 2000 316420 is related to transgene pigs obtained by methods involving micro injection and not sperm mediated gene transfer. The pig may carry a gene causing an autosomal dominant disease.

However as pigs constitute a distinct evolutionary clade in comparison with humans the introduction of mutations known as disease causing mutations in specific genes in humans cannot be expected to yield a desired phenotype in the pig model.

There is a need for improved animal models for human diseases in order to gain more information of the onset progression and treatment regimes of hereditary diseases in humans.

Until now it has been believed that the phenotypic display of an autosomal dominant disease is caused by the continuous expression of an inherited mutated gene. However the present invention discloses that a phenotype of an autosomal dominant disease is caused by a sufficiently high expression of the mutated gene in a transient manner. Thus the present invention discloses that the expression of a gene involved in the development of autosomal dominant diseases primarily has to be expressed in sufficiently high amounts at a specific time point during the development of the embryo of a non human animal model. The fact that the expression of particular genes associated with autosomal dominant genes is transient also allows for the production of non human animal models by the addition of gene products to for example embryos or other target cells seeding effect .

In a first aspect the present invention relates to non human animal model for a hereditary autosomal dominant disease wherein the non human animal model expresses at least one phenotype associated with said hereditary autosomal dominant disease obtained by a genetic determinant.

In a second aspect a non human animal model for a hereditary autosomal dominant disease wherein the non human animal model expresses at least one phenotype associated with said hereditary autosomal dominant disease obtained by sperm mediated gene transfer.

A third aspect of the present invention pertains to a pig model for a hereditary autosomal dominant disease obtained by a genetic determinant wherein the pig model expresses at least one phenotype associated with said hereditary autosomal disease.

A fourth aspect relates to a pig model for a hereditary autosomal dominant disease obtained by a genetic determinant wherein said disease is a protein conformation disease.

A fifth aspect concerns a pig model for a hereditary neurodegenerative autosomal dominant disease obtained by a genetic determinant.

In further aspects of the invention is disclosed a pig model for amyotrophic lateral sclerosis Alzheimer s Disease Parkinson s Disease diseases related to Trinucleotide Repeats Huntington s chorea or dyschondroplasia obtained by a genetic determinant wherein the pig model expresses at least one phenotype associated with Amyotrophic Lateral Sclerosis Alzheimer s Disease Parkinson s Disease diseases related to Trinucleotide Repeats Huntington s chorea or dyschondroplasia respectively.

The present invention also relates to a method for producing the model according to the present invention said method comprising the steps of

i providing semen from a male non human animal ii providing at least one genetic determinant capable of establishing said at least one phenotype associated with said hereditary disease when the at least one genetic determinant is expressed in said non human animal model iii contacting said semen and said at least one genetic determinant iv fertilising an oocyte from a female non human animal with the semen and the genetic material and v incubating said fertilised oocyte under conditions allowing said fertilised oocyte to develop into said non human animal model.

Furthermore a method for evaluating the response of a therapeutical treatment of a hereditary disease said method comprising the steps of a. providing the non human animal model according to the present invention b. treating said non human animal with at least one pharmaceutical composition exerting an effect on said at least one phenotype and c. evaluating the effect observed is disclosed.

Yet other aspects concern a non human sperm cell comprising at least one genetic determinant exerting at least one dominant phenotype for at least one hereditary disease when expressed in a non human animal host organism and a method for producing the non human sperm cell said method comprising the steps of a. providing a non human sperm cell b. providing at least one genetic determinant exerting a dominant phenotype for a hereditary disease when expressed in a non human animal host organism c. contacting said non human sperm cell and said at least one genetic determinant wherein said contacting results in the uptake of the genetic determinant into the non human sperm cell.

Moreover the present invention also relates to a composition comprising a non human sperm cell in combination with at least one genetic determinant exerting at least one dominant phenotype for at least one hereditary disease when expressed in a non human animal host organism.

In further aspects the invention pertains to a method for fertilising an oocyte by sperm mediated gene transfer said method comprising the steps of providing the non human sperm cell or the composition as defined above and introducing said non human sperm cell into the oocyte to be fertilised.

The present invention also relates to an embryo obtained by fertilising an oocyte with the non human sperm cell or with the composition as defined herein.

Yet a further aspect relates to a method for the cultivation and development of the embryo as described herein said method comprising the step of cultivating said embryo under conditions allowing the embryo to develop into a non human animal offspring expressing said genetic determinant and exerting a dominant phenotype for a hereditary disease.

In a final aspect is disclosed a method for screening the efficacy of a pharmaceutical composition said method comprising the steps of a. providing the non human animal model of the present invention b. expressing in said animal model said at least one genetic determinant and exerting said dominant phenotype for said hereditary disease c. administering to said non human animal the pharmaceutical composition the efficacy of which is to be evaluated and d. evaluating the effect if any of the pharmaceutical composition on the phenotype exerted by the genetic determinant when expressed in the non human model.

The term sperm is used to refer to a male gamete cell and includes without limitation spermatogonia primary and secondary permatocytes spermatids differentiating spermatids round spermatids and spermatozoa.

The term oocyte is used to refer to a female gamete cell and includes primary oocytes secondary oocytes and mature unfertilized ovum.

In some cases the term embryo is used to describe a fertilized oocyte after implantation in the uterus until 8 weeks after fertilization at which stage it becomes a foetus. According to this definition the fertilized oocyte is often called a pre embryo until implantation occurs. However throughout this patent application we will use a broader definition of the term embryo which includes the pre embryo phase. It thus encompasses all developmental stages from the fertilization of the oocyte through morula blastocyst stages hatching and implantation. An embryo is approximately spherical and is composed of one or more cells blastomeres surrounded by a gelatine like shell the acellular matrix known as the zona pellucida. The zona pellucida performs a variety of functions until the embryo hatches and is a good landmark for embryo evaluation. An embryo is formed when an oocyte is fertilized by fusion or injection of a sperm cell spermatozoa . The feritilised oocyte is traditionally called an embryo for the first 8 weeks. After that i.e. after eight weeks and when all major organs have been formed it is called a foetus. However the distinction between embryo and foetus is not generally well defined. During embryonic development blastomere numbers increase geometrically 1 2 4 8 16 etc. . Synchronous cell division is generally maintained to the 16 cell stage in embryos. After that cell division becomes asynchronous and finally individual cells possess their own cell cycle. At about the 32 cell stage morula stage embryos undergo compaction as inter cell adhesion occur when adhesion proteins are expressed.

Accordingly the term embryo is used in the following to denote each of the stages fertilized oocyte zygote 2 cell 4 cell 8 cell 16 cell morula blastocyst expanded blastocyst and hatched blastocyst as well as all stages in between e.g. 3 cell or 5 cell .

The term non human animal can be a non human primate e.g. an ape or a monkey and a farm animal such as an animal selected from the group consisting of cattle swine sheep goats horses and donkeys. Accordingly the non human animal can e.g. be a cow a bull a bison a buffalo a pig a big horn sheep a pony a mule a deer an elk a lama and an alpaca. Similarly the non human animal can be a rodent such as a mouse or rat.

The term non human animal model refers to any non human animal in which one or more cells comprise genetic determinants. The non human animal comprising genetic determinants may for example be the result of introduction of the genetic determinant by sperm mediated gene transfer. However the genetic determinant may also be introduced for example by injection into cells or tissues desired.

The term genetic determinant of the present invention refers to genes or parts thereof transcriptional products or parts thereof and or translational products or part thereof that confer the display of one or more features of phenotypes of autosomal dominant hereditary diseases. Thus in some embodiments the term genetic determinant is used herein to refer to a single stranded or double stranded polynucleotide molecule or nucleic acid comprising a structural gene of interest. The genetic determinant encodes a protein not ordinarily made in appreciable amounts in the target cells.

The term genetic determinant is also used to refer to a single stranded or double stranded ribonucleic acid RNA expressed from a gene of interest. Thus genetic determinants include nucleic acids which are not ordinarily found in the genome of the target cell. Genetic determinants also include nucleic acids which are ordinarily found within the genome of the target cell but is in a form which allows for the expression of proteins which are not ordinarily expressed in the target cells in appreciable amounts. Alternatively genetic determinants may encode a variant or mutant form of a naturally occurring protein.

The term genetic determinant is also used herein to refer to a protein or part thereof or a RNA molecule or part thereof of a gene of interest wherein the gene or polynucleotide encoding said protein or RNA is not present in the target cell. Throughout the description and claims either the three letter code or the one letter code for natural amino acids are used. Where the L or D form has not been specified it is to be understood that the amino acid in question has the natural L form cf. Pure Appl. Chem. Vol. 56 5 pp 595 624 1984 or the D form so that the peptides formed may be constituted of amino acids of L form D form or a sequence of mixed L forms and D forms.

Where nothing is specified it is to be understood that the C terminal amino acid of a peptide for use according to the invention exists as the free carboxylic acid this may also be specified as OH . However the C terminal amino acid of a peptide for use according to the invention may be the amidated derivative which is indicated as NH . Where nothing else is stated the N terminal amino acid of a polypeptide comprises a free amino group this may also be specified as H .

A peptide fragment homologue or variant for use according to the invention can also comprise one or several unnatural amino acids.

Conservative amino acid substitutions Substitutions within the groups of amino acids shown below are considered conservative amino acid substitutions. Substitutions between the different groups of amino acids are considered non conservative amino acid substitutions.

In one embodiment the genetic determinant is in the form of microRNAs miRNA that are single stranded RNA molecules of about 21 23 nucleotides in length. miRNAs are typically encoded by genes that are transcribed from DNA but not translated into protein non coding RNA instead they are processed from primary transcripts known as pri miRNA to short stem loop structures called pre miRNA and finally to functional miRNA. Mature miRNA molecules are partially complementary to one or more messenger RNA. Thus protein or RNA may be produced outside the target cell and introduced into a target cell by any method known to the skilled person. For example the genetic determinant is provided from extracts of brains from diseased subjects such as humans or animals of the present invention. Subsequently said brain tissue is introduced into a target cell of the present invention.

The target cell may be a sperm cell an oocyte a fertilized oocyte an embryo which includes the pre embryo phase encompassing all developmental stages from the fertilization of the oocyte through morula blastocyst stages hatching and implantation a fetus or a cell derived from tissue of the developing fetus. In one embodiment of the present invention the target cell is an adult animal or tissue thereof.

The genetic determinant may be introduced to the target cell by for example injection virus mediated transfer or similar methods known to the skilled person.

The genetic determinant may in the form of RNA or protein be introduced into the target cell to yield a non human animal model for autosomal dominant diseases such as neurodegenerative diseases as described elsewhere herein for example protein conformation disorders. In preferred embodiments the genetic determinant is introduced into the target cell to produce a pig model expressing at least one phenotype associated with ALS Alzheimer s disease Parkinson s disease Huntington s chorea diseases related to trinucleotide repeats. Similarly in one particular embodiment a pig model is produced expressing at least one phenotype associated with dyschondroplasia.

The genetic determinant may in particular embodiments be introduced by sperm mediated gene transfer to produce a pig model for autosomal dominant diseases such as neurodegenerative diseases as described elsewhere herein for example protein conformation disorders. In preferred embodiments the genetic determinant is introduced by sperm mediated gene transfer to produce a pig model expressing at least one phenotype associated with ALS Alzheimer s disease Parkinson s disease Huntington s chorea diseases related to trinucleotide repeats. Similarly in one particular embodiment a pig model is produced expressing at least one phenotype associated with dyschondroplasia.

The genetic determinant when present as a gene or DNA construct need not be integrated into the genome of the target cell fertilised oocyte embryo fetus or tissue. In such an example the gene or DNA construct needs to be expressed in an amount sufficient for triggering a cascade that eventually results in the onset and progression of the disease in question. This is in particular the case for autosomal diseases of the neurodegenerative kind such as protein conformation diseases. Thus the genetic determinant involved in ALS Alzheimer s Disease Parkinson s disease diseases related to trinucleotide repeats and Huntington s chorea need not to be integrated in the genome of the pig model. The expression of the gene of interest or DNA construct need not be continuous but has to take place during the development of the embryo to fetus. Similarly when the genetic determinant is present in the form of protein where the protein is involved in the development of protein conformation diseases the protein need to be present in the embryo or fetus but need not be present continuously in the pig model.

The genetic determinant when present in the form of DNA or cDNA may further comprise regulatory sequences to direct expression of the DNA or cDNA. Such regulatory sequences may be promoters or enhancers. The term promoter will be used here to refer to a group of transcriptional control modules that are clustered around the initiation site for RNA polymerase II. Promoters are composed of discrete functional modules each consisting of approximately 7 20 bp of DNA and containing one or more recognition sites for transcriptional activator proteins. At least one module in each promoter functions to position the start site for RNA synthesis. The best known example of this is the TATA box.

Additional promoter elements regulate the frequency of transcriptional initiation. Typically these are located in the region 30 110 bp upstream of the start site although a number of promoters have recently been shown to contain functional elements downstream of the start site as well. The spacing between elements is flexible so that promoter function is preserved when elements are inverted or moved relative to one another. Enhancers were originally detected as genetic elements that increased transcription from a promoter located at a distant position on the same molecule of DNA. The basic distinction between enhancers and promoters is operational. An enhancer region as a whole must be able to stimulate transcription at a distance this need not be true of a promoter region or its component elements. On the other hand a promoter must have one or more elements that direct initiation of RNA synthesis at a particular site and in a particular orientation whereas enhancers lack these specificities. Aside from this operational distinction enhancers and promoters are very similar entities. They have the same general function of activating transcription in the cell. They are often overlapping and contiguous often seeming to have a very similar modular organization.

The terms polynucleotide and nucleic acid are used interchangeably and when used in singular or plural generally refers to any polyribonucleotide or polydeoxyribonucleotide which may be unmodified RNA or DNA or modified RNA or DNA. Thus for instance polynucleotides as defined herein include without limitation single and double stranded DNA DNA including single and double stranded regions single and double stranded RNA and RNA including single and double stranded regions hybrid molecules comprising DNA and RNA that may be single stranded or more typically double stranded or include single and double stranded regions. In addition the term polynucleotide as used herein refers to triple stranded regions comprising RNA or DNA or both RNA and DNA. The strands in such regions may be from the same molecule or from different molecules. The regions may include all of one or more of the molecules but more typically involve only a region of some of the molecules. One of the molecules of a triple helical region often is an oligonucleotide. The term polynucleotide specifically includes cDNAs. The term includes DNAs including cDNAs and RNAs that contain one or more modified bases. Thus DNAs or RNAs with backbones modified for stability or for other reasons are polynucleotides as that term is intended herein. Moreover DNAs or RNAs comprising unusual bases such as inosine or modified bases such as tritiated bases are included within the term polynucleotides as defined herein. In general the term polynucleotide embraces all chemically enzymatically and or metabolically modified forms of unmodified polynucleotides as well as the chemical forms of DNA and RNA characteristic of viruses and cells including simple and complex cells.

 Sperm mediated gene transfer as used herein refers to any method wherein a non human animal sperm cell is mixed with a genetic determinant gene under conditions resulting in the genetic determinant being 1 taken up by the sperm cell 2 occurring as extrachromosomal DNA or as stably integrated into the genetic material genome harboured by the sperm cell and 3 optionally expressed in said sperm cell. Once taken up by the sperm cell the genetic material gene can be transferred to a non human animal model which allows one to study the expression of the genetic determinant in the chosen genetic background.

 Autosomal diseases are used herein to refer to diseases which are inherited through the non sex chromosomes pairs 1 through 22 . The term autosomal dominant is used herein to refer to a single abnormal gene on one of the autosomal chromosomes one of the first 22 non sex chromosomes from either parent which can cause certain diseases. One of the parents will usually have the disease as the gene is dominant in this mode of inheritance. Only one parent must have an abnormal gene in order for the offspring to inherit the disease.

The present invention relates to animal models for hereditary autosomal dominant diseases. The models of the present invention can be used as model for hereditary autosomal dominant diseases as known from humans. Animal models for human diseases and especially those diseases such as the autosomal dominant diseases which develop over a long period of time having a late onset in life are very useful in order to gain information on the onset progression and treatment regime of individuals suffering from such type of diseases. It is appreciated that the animal models are non human animals. In one aspect of the invention the non human animal model for a hereditary autosomal dominant disease is a model wherein the non human animal model expresses at least one phenotype associated with said hereditary autosomal dominant disease obtained by at least one genetic determinant. In another aspect of the invention the non human animal model for a hereditary autosomal dominant disease is a model wherein the non human animal model expresses at least one phenotype associated with said hereditary autosomal dominant disease obtained by sperm mediated gene transfer.

It is appreciated that the non human animal model may be obtained by the use of at least one two three four five six seven eight nine or ten genetic determinants. The genetic determinants of a given autosomal dominant disease whether in the form of DNA RNA or protein as described herein may be combined in one animal model in order to obtain a strong phenotype of an autosomal dominant disease.

Autosomal dominant diseases are as described elsewhere herein diseases that are normally inherited through the non sex chromosomes from one of the parents. Autosomal dominant diseases comprise diseases known as protein conformation diseases neurodegenerative diseases such as amyotrophic lateral sclerosis ALS Alzheimer s disease Parkinson s disease diseases related to Trinucleotide Repeats for example huntington s chorea but also conditions related to dyschondroplasia.

Protein conformation diseases are one type of neurodegenerative diseases in which protein folding disorders occur. The protein misfolding of specific proteins can be observed in affected neuronal tissue. Protein conformation diseases according to the present invention are ALS Alzheimer s disease Parkinson s disease diseases associated with trinucleotide repeats and Huntington s chorea.

In one embodiment the non human animal model expresses at least one phenotype associated with ALS. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with ALS.

 Amyotroph lateral sclerosis ALS is used herein to refer to any neurodegenerative disease that usually attacks both upper and lower motor neurons and causes degeneration throughout the brain and spinal cord.

Physicians have limited choices for treating ALS. At this time Riluzole is the only drug that has been approved by the FDA for treatment of ALS. In clinical trials Riluzole has shown a slight benefit in modestly increasing survival time of patients suffering from ALS.

Amyotrophic lateral Sclerosis ALS is the most common motor neurodegenerative disease characterised by a progressive loss of motor neurons in the spinal cord brain stem and motor cortex causing weakness muscular wasting and paresis ultimately leading to death. Symptoms of the disease constitute weakness stiffness abnormal reflexes fasciculations cramps and atrophy. However especially in the early phases of the disease ALS can be difficult to diagnose since all symptoms are rarely present at that stage 1 .

A common first symptom is a painless weakness in a hand foot arm or leg which occurs in more than half of all cases. Other early symptoms include swallowing or walking difficulty. The biological mechanisms that cause ALS are only partially understood. One known cause of ALS is a mutation of a specific human gene The SOD1 gene. This mutation is believed to make a defective protein that is toxic to motor nerve cells. The SOD1 mutation however accounts for only 1 or 2 percent of ALS cases or 20 percent of the familial inherited cases. Familial ALS represents between five to 10 percent of all cases the remaining cases seemingly arise spontaneously and attacks previously healthy adults.

The present invention relates to the production of a non human animal model for ALS. The below Table 1 shows mutations which are associated with ALS in humans. According to the present invention any of the in table 1 listed mutations and substitutions as known from humans are embodiments of the present invention. Thus the mutated forms of the porcine homolog of the human SOD1 and or human SOD1 genes cDNA RNA and or protein or parts thereof may comprise any of the listed mutations in order to produce an non human animal model for ALS and in particular a pig model for ALS. It is appreciated that at least one or more mutations may be introduced into the porcine homolog of the human SOD1 and or human SOD1 genes cDNA RNA and or protein or parts thereof. Any number of the listed mutations may be used in combination.

Furthermore two mutations in ALS 2 Ala46delA and Leu623delCT and three mutations in ALS4 SETX Thr3Ile Leu389Ser and Arg2136His are associated with ALS.

Thus in one aspect of the present invention the non human animal model expressing at least one phenotype associated with ALS due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human SOD1 gene mRNA and or protein see SEQ ID NO 1 SEQ ID NO 2 comprises at least one mutation yielding the acid mutations as described in table 1 or as described above for ALS 2 Ala46delA and Leu623delCT and three mutations in ALS4 SETX Thr3Ile Leu389Ser and Arg2136His all associated with ALS.

Similarly in one embodiment the non human animal model expressing at least one phenotype associated with ALS due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine SOD1 gene mRNA and or protein see SEQ ID NO 3 SEQ ID NO 4 or fragments or parts thereof comprising at least one mutation yielding the amino acid mutations as described in table 1 or as described above for ALS 2 Ala46delA and Leu623delCT and three mutations in ALS4 SETX Thr3Ile Leu389Ser and Arg2136His all associated with ALS. It is appreciated that the at least one mutation is present in the gene fragment or DNA fragment RNA fragment or protein part. In particular the mutated porcine SOD1 cDNA with SEQ ID NO 5 is used to produce a pig model for ALS.

The non human animal model for ALS may be generated by introduction of said at least one genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for ALS is generated by sperm mediated gene transfer.

The present invention discloses a pig model for ALS which in one embodiment is produced by sperm mediated gene transfer of the mutated porcine SOD1 SEQ ID NO 5 as described elsewhere herein. However the pig model for ALS may also be produced by introducing the mutated procine SOD1 SEQ ID NO 5 or protein expressed SEQ ID NO 6 thereof into a target cell.

It is appreciated that at least one mutation yielding the amino acid mutation of SOD1 in the human SOD1 as listed in table 1 or as described above for ALS 2 Ala46delA and Leu623delCT and three mutations in ALS4 SETX Thr3Ile Leu389Ser and Arg2136His all associated with ALS are comprised in the human or porcine SOD1 gene or DNA RNA or proteins of the present invention such as for example at least two mutations at least three mutations at least four mutations at least five mutations at least six mutations at least seven mutations at least eight mutations at least ten mutations at least fifteen mutations yielding the amino acid mutation of SOD1 of table 1 or as described above for ALS 2 Ala46delA and Leu623delCT and three mutations in ALS4 SETX Thr3Ile Leu389Ser and Arg2136His all associated with ALS.

The non human animal model for ALS in particular the pig model for ALS will typically develop at least one of the symptoms described above such as weakness stiffness abnormal reflexes fasciculations cramps and atrophy. A common first symptom is a painless weakness in a leg which occurs in more than half of all cases. Other early symptoms include swallowing or walking difficulty. Furthermore histopathology performed on a biopsy as described in the examples herein can be used to diagnose the animal model with ALS. The analysis of the motor neurons for the presence aggregates or even loss of the motor neurons are also characteristics of ALS.

In another aspect of the present invention the non human animal model expresses at least one phenotype associated with Alzheimer s disease. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with Alzheimer s disease.

Alzheimer s disease is used herein to refer to any neurodegenerative brain disorder characterized by progressive memory loss and severe dementia in advanced cases. Alzheimer s disease is associated with certain abnormalities in brain tissue involving a particular protein beta amyloid. Memory impairment is a necessary feature for the diagnosis of this type of dementia. Change in one of the following areas must also be present language decision making ability judgment attention and other areas of mental function and personality.

The rate of progression is different for each person. If Alzheimer s disease develops rapidly it is likely to continue to progress rapidly. If it has been slow to progress it will likely continue on a slow course. There are two types of Alzheimer s disease early onset and late onset. In early onset Alzheimer s disease symptoms first appear before age 60. Early onset Alzheimer s disease is much less common accounting for only 5 10 of cases. However it tends to progress rapidly.

Early onset disease can run in families and involves autosomal dominant inherited mutations that may be the cause of the disease. So far three early onset genes have been identified. Late onset Alzheimer s disease the most common form of the disease develops in people 60 and older and is thought to be less likely to occur in families. Late onset Alzheimer s disease may run in some families but the role of genes is less direct and definitive. These genes may not cause the problem itself but simply increase the likelihood of formation of plaques and tangles or other Alzheimer s disease related pathologies in the brain.

The cause of Alzheimer s disease is not entirely known but is thought to include both genetic and environmental factors. A diagnosis of Alzheimer s disease is made based on characteristic symptoms and by excluding other causes of dementia. The only way to validate a case of Alzheimer s disease is by microscopic examination of a sample of brain tissue after death.

The brain tissue shows neurofibrillary tangles twisted fragments of protein within nerve cells that clog up the cell neuritic plaques abnormal clusters of dead and dying nerve cells other brain cells and protein and senile plaques areas where products of dying nerve cells have accumulated around protein . Although these changes occur to some extent in all brains with age there are many more of them in the brains of people with Alzheimer s disease.

The destruction of nerve cells neurons leads to a decrease in neurotransmitters substances secreted by a neuron to send a message to another neuron . The correct balance of neurotransmitters is critical to the brain. By causing both structural and chemical problems in the brain Alzheimer s disease appears to disconnect areas of the brain that normally work together.

The below indicated substitutions are believed to be relevant regarding transgenic porcine models for Alzheimer s disease.

Thus in one embodiment the non human animal model expressing at least one phenotype associated with Alzheimer s disease due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human PSEN1 gene mRNA and or protein see SEQ ID NO 7 SEQ ID NO 8 comprises at least one mutation yielding the amino acid mutations as described in table 4.

Similarly in one embodiment the non human animal model expressing at least one phenotype associated with Alzheimer s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine PSEN1 gene mRNA and or protein see SEQ ID NO 9 SEQ ID NO 10 or fragments or parts thereof respectively comprising at least one mutation yielding the amino acid mutations as described in table 4 It is appreciated that the at least one mutation is present in the gene fragment or DNA fragment RNA fragment or protein part. In particular the mutated porcine PSEN1 protein with SEQ ID NO 11 is used to produce a pig model for Alzheimer s disease.

The non human animal model for ALS may be generated by introduction of said at least one genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for Alzheimer s disease is generated by sperm mediated gene transfer.

According to the present invention a pig model for Alzheimer s disease is in one embodiment produced by sperm mediated gene transfer of the mutated porcine PSEN1 as described elsewhere herein. However the pig model for Alzheimer s disease may also be produced by introducing the mutated PSEN1 or protein expressed thereof SEQ ID NO 11 into a target cell.

Moreover in one embodiment the non human animal model expressing at least one phenotype associated with Alzheimer s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human PSEN2 gene mRNA and or protein see SEQ ID NO 12 SEQ ID NO 13 comprises at least one mutation yielding the amino acid mutations as described in table 3.

Similarly in one embodiment the non human animal model expressing at least one phenotype associated with Alzheimer s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine PSEN2 gene mRNA and or protein corresponding to SEQ ID NO 14 SEQ ID NO 15 or fragments or parts thereof respectively fitted with at least one mutation yielding the amino acid mutations as described in table 3. It is appreciated that the at least one mutation is present in the gene fragment or DNA fragment RNA fragment or protein part. In particular the mutated porcine PSEN2 DNA with SEQ ID NO 16 is used to produce a pig model for Alzheimer s disease.

The non human animal model for ALS may be generated by introduction of said at least one genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for Alzheimer s disease is generated by sperm mediated gene transfer.

According to the present invention a pig model for Alzheimer s disease is in one embodiment produced by sperm mediated gene transfer of the mutated porcine PSEN1 as described elsewhere herein. However the pig model for Alzheimer s disease may also be produced by introducing the mutated PSEN1 or protein expressed thereof into a target cell SEQ ID NO 16 .

Furthermore in yet another embodiment the non human animal model expressing at least one phenotype associated with Alzheimer s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human APP gene mRNA and or protein SEQ ID NO 17 SEQ ID NO 18 comprises at least one mutation yielding the amino acid mutations as described in table 2. Similarly in one embodiment the non human animal model expressing at least one phenotype associated with Alzheimer s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine APP gene mRNA and or protein corresponding to SEQ ID NO 19 SEQ ID NO 20 or fragments or parts thereof comprising at least one mutation yielding the amino acid mutations as described in table 2. It is appreciated that the at least one mutation is present in the gene fragment or cDNA fragment RNA fragment or protein part. In particular the mutated porcine APP DNA is used to produce a pig model for Alzheimer s disease.

The non human animal model for Alzheimer s disease may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell.

In one particular embodiment the non human animal model for Alzheimer s disease is generated by sperm mediated gene transfer.

According to the present invention a pig model for Alzheimer s disease is in one embodiment produced by sperm mediated gene transfer of the mutated porcine APP as described elsewhere herein. However the pig model for Alzheimer s disease may also be produced by introducing the mutated APP or protein expressed thereof into a target cell SEQ ID NO 21 .

It is within the scope of the present invention that at least one mutation yielding the amino acid mutation homologous to the human PSEN1 PSEN2 and or APP as listed in table 4 3 and or 2 is comprised in the human or porcine PSEN1 PSEN2 and or APP gene or DNA RNA or proteins of the present invention such as for example at least two mutations at least three mutations at least four mutations at least five mutations at least six mutations at least seven mutations at least eight mutations at least ten mutations at least fifteen mutations yielding the amino acid mutation homologous to the human PSEN1 PSEN2 and or APP as listed in table 4 3 and or 2. It is appreciated that one or more of the genes and mutations thereof may be combined in an animal model for example a pig model for Alzheimer s disease.

The non human animal model for Alzheimer s disease in particular the pig model for Alzheimer s disease will typically develop at least one of the symptoms described above such as progressive memory loss and severe dementia in advanced cases which can be monitored by behavioural studies. Evidence also exists for the impairment of olfactoric sense which can be monitored by behavioural changes. Furthermore scannings of the brain of the animal models by magnetic resonance and or positron emission tomography can also be employed to determine whether the animal model is indicative for Alzheimer s disease.

In one embodiment of the present invention the non human animal model expresses at least one phenotype associated with Parkinson s disease. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with Parkinson s disease.

 Parkinson s disease is used herein to refer to an inherited condition usually associated with the following symptoms all of which result from the loss of dopamine producing brain cells tremor or trembling of the arms jaw legs and face stiffness or rigidity of the limbs and trunk bradykinesia slowness of movement postural instability or impaired balance and coordination.

Parkinson s disease PD is a common progressive neurodegenerative disease affecting 1 2 of the population over 60 years of age. The number of PD affected individuals reaches a maximum value between 70 and 79 years of age with a mean age of onset between 60 and 65 years 1 . The cardinal clinical symptoms of PD are resting tremor bradykinesia rigidity and postural instability. The pathological manifestations of PD are characterized by degeneration of dopaminergic neurons in the substantia nigra pars compacta and the presence of so called Lewy bodies in degenerating neurons. Lewy bodies are cytoplasmic protein inclusions and the main constituent is the protein synuclein. Lewy bodies are required for a pathological diagnosis of PD but can also be observed in other neurodegenerative diseases. Degeneration of dopaminergic neurons results in decreased production of dopamine and the lack of this signal substance is responsible for bradykinesia and rigidity. The etiology of PD is largely unknown but is probably due to both genetic and environmental factors. Impaired mitochondrial function changes in protein sorting by the ubiquitin proteasome pathway and facilitated apoptosis may all represent factors associated with development of PD 2 . Environmental factors suggested are exposure to pesticides and heavy metals 3 .

Several transgenic animal models of PD have been established including nematodes mice and rats and different PD associated genes. Several limitations to the mice models have been observed including the absence of loss of nigrostriatal dopaminergic neurons in some of the models suggesting that a better animal model would be of advantage regarding the study of the factors involved in PD pathology and underlying mechanisms.

The below indicated substitutions are believed to be relevant regarding transgenic porcine models for Parkinsons s disease.

One aspect of the present invention thus relates to a non human animal model expressing at least one phenotype associated with Parkinson s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human SNCA gene mRNA and or protein SEQ ID NO 22 SEQ ID NO 23 comprises at least one mutation yielding the amino acid mutations as described in table 5.

Similarly in one embodiment the non human animal model expressing at least one phenotype associated with Parkinson s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine SNCA gene mRNA and or protein corresponding to SEQ ID NO 24 SEQ ID NO 25 or fragments or parts thereof respectively comprising at least one mutation yielding the amino acid mutations as described in table 5. It is appreciated that the at least one mutation is present in the gene fragment or DNA fragment RNA fragment or protein part. In particular the mutated porcine SNCA cDNA with SEQ ID NO 26 is used to produce a pig model for Parkinson s disease.

The non human animal model for ALS may be generated by introduction of said at least one genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for Parkinson s disease is generated by sperm mediated gene transfer.

According to the present invention a pig model for Parkinson s disease is in one embodiment produced by sperm mediated gene transfer of the mutated porcine SNCA SEQ ID NO 26 as described elsewhere herein. However the pig model for Parkinson s disease may also be produced by introducing the mutated SNCA DNA SEQ ID NO 26 or protein expressed thereof into a target cell SEQ ID NO 27 .

Moreover in one embodiment the non human animal model expressing at least one phenotype associated with Parkinson s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human UCHL1 gene mRNA and or protein SEQ ID NO 28 SEQ ID NO 29 comprises at least one mutation yielding the amino acid mutations as described in table 6.

Similarly in one embodiment the non human animal model expressing at least one phenotype associated with Parkinson s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine UCHL1 gene mRNA and or protein or fragments or parts thereof comprising at least one mutation yielding the amino acid mutations as described in table 6. It is appreciated that the at least one mutation is present in the gene fragment or DNA fragment RNA fragment or protein part. In particular the mutated porcine UCHL1 DNA is used to produce a pig model for Parkinson s disease.

The non human animal model for Parkinson s disease may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell.

In one particular embodiment the non human animal model for Parkinson s disease is generated by sperm mediated gene transfer.

According to the present invention a pig model for Parkinson s disease is in one embodiment produced by sperm mediated gene transfer of the mutated porcine UCHL1 as described elsewhere herein. However the pig model for Parkinson s disease may also be produced by introducing the mutated UCHL1 or protein expressed thereof into a target cell.

Furthermore in yet another embodiment the non human animal model expressing at least one phenotype associated with Parkinson s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human LRRK2 gene mRNA and or protein SEQ ID NO 30 SEQ ID NO 31 comprises at least one mutation yielding the amino acid mutations as described in table 7.

Similarly in one embodiment the non human animal model expressing at least one phenotype associated with Parkinson s disease due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine LRRK2 gene mRNA and or protein or fragments or parts thereof respectively fitted with at least one mutation yielding the amino acid mutations as described in table 7. It is appreciated that the at least one mutation is present in the gene fragment or DNA fragment RNA fragment or protein part. In particular the mutated porcine LRRK2 DNA is used to produce a pig model for Parkinson s disease.

The non human animal model for Parkinson s disease may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell.

In one particular embodiment the non human animal model for Parkinson s disease is generated by sperm mediated gene transfer.

According to the present invention a pig model for Parkinson s disease is in one embodiment produced by sperm mediated gene transfer of the mutated porcine LRRK2 as described elsewhere herein. However the pig model for Alzheimer s disease may also be produced by introducing the mutated LRRK2 or protein expressed thereof into a target cell.

It is within the scope of the present invention that at least one mutation yielding the amino acid mutation homologous to the human SNCA UCHL1 and or LRRK2 as listed in table 5 6 and or 7 is comprised in the human or porcine SNCA UCHL1 and or LRRK2 gene or DNA RNA or proteins of the present invention such as for example at mutations at least six mutations at least seven mutations at least eight mutations at least ten mutations at least fifteen mutations yielding the amino acid mutation homologous to the human SNCA UCHL1 and or LRRK2 as listed in table 5 6 and or 7. It is appreciated that one or more of the genes and mutations thereof may be combined in an animal model for example a pig model for Parkinson s disease.

The non human animal model for Parkinson s disease in particular the pig model for Parkinson s disease will typically develop at least one of the symptoms described above such as symptoms all of which result from the loss of dopamine producing brain cells tremor or trembling of the jaw legs and face stiffness or rigidity of the limbs and trunk bradykinesia slowness of movement postural instability or impaired balance and coordination.

In one embodiment of the present invention the non human animal model expresses at least one phenotype associated with diseases related to trinucleotide repeat disorders. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with diseases related to trinucleotide repeat disorders.

Trinucleotide repeat disorders or expansion disorders are caused by stretches of DNA in a gene that contain the same trinucleotide sequence repeated many times. Unstable microsatellite repeats are found throughout all genomic sequences and TNRs constitute a subset of such unstable repeats.

Nucleotide repeat instability is associated with more than 40 inherited neurodegenerative neuromuscular and mental retardation disorders in humans 2 3 . The nucleotide repeat instability process is a dynamic process where mutations continue to recur during meiosis and in mitotic tissue 3 . Long stretches of repeats are more likely to expand than short stretches of repeats and the length is correlated with age of onset and the severity of disease a phenomena called anticipitation 3 . Diseases caused by trinucleotide repeat TNR instability can be divided into two groups. In the first group the TNRs reside in the untranslated part of the affected genes. The untranslated TNR expansions constituting either CTGs CAGs GAAs or CGGs result in an RNA gain of function that may alter the gene expression control of the affected genes or have cis and trans effects on splicing and gene regulation at the chromatin level 4 7 . Expanded noncoding TNRs have been identified as causative mutations in disorders including Friedreich ataxia spinocerebellar ataxia 8 SCA 8 SCA 12 myotonic dystrophy and fragile X syndromes 3 .

In the second group the TNRs are located in the coding region of the transcript. This type of TNRs are translated in frame of the coding region and expansions include GAC encoding poly aspargine GCG encoding poly alanine and the most commonly identified CAG TNRs encoding poly glutamine 3 . Expanded CAG TNRs are identified as causative mutations in disorders including SCA 1 SCA 2 SCA 3 SCA 6 SCA 7 SCA17 Huntington s disease spinal and bulbar muscular atrophy SBMA and dentatorubral pallidoluysian atrophy DRPLA . In these disorders cytoskeletal and vesicular functions are affected as well as the regulation of cellular gene expression due to the sequestering of transcriptional regulatory proteins.

The molecular mechanisms responsible for TNR instability are not completely elucidated. The degree of tissue specific and inherited TNR instability is determined by both the specific cis sequences within the affected genes and trans functioning metabolic proteins as for example DNA repair proteins 3 . TNR instability probably involves the formation of specific DNA structures during DNA replication repair and recombination 8 . Slippage during DNA replication is the best characterized mechanism. The direction of DNA replication through the TNR tract also affects the stability 9 11 . In addition to the TNR itself the sequence environment of the repeat contributes to the mechanism of instability and for example similar CAG tract lengths show different stability depending on the genomic context 3 12 . In the human population the length of TNR repeat tracts is polymorphic but stably transmitted 13 . Beyond a certain TNR length which appears to be gene specific the TNR tract becomes unstable 3 . An unstable TNR tract that has not yet expanded to a size sufficient for the full disease phenotype is called a premutation 2 . However TNR expansions of the premutation size in for example FMR1 have been shown to result in a specific disease phenotype of late onset. The genetic instability of TNRs is related to the repeat tract length and expanded tracts have increased risk of being affected by a subsequent expanding mutation than the original tract 2 3 14 . Also TNR interruptions as for example CAA triplets in CAG TNRs play an important role conferring TNR stability and their absence predisposes alleles towards instability and pathological expansions 15 . A large level of somatic and transmitted instability is observed for premutation and fully expanded TNR tracts 2 3 . The fully expanded and disease causing TNR tracts can be composed of approximately twenty TNRs as observed in SCA 6 to several thousands as observed in for example myotonic dystrophy 2 3 . Interestingly it was recently shown that an induced level of transcription promotes contraction of TNR tracts in human cells 16 . Furthermore transgenic animal models of TNR instability also points out the importance for specific TNR flanking sequences to create TNR instability 9 17 18 .

One embodiment of the present invention relates to a non human animal model expressing at least one phenotype associated with myotonic dystrophy. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with myotonic dystrophy.

Myotonic dystrophy is caused by TNR in the 3 UTR of the human myotonic dystrophy protein kinase gene DMPK where a CTG TNR is located 19 23 . The normal size of this TNR varies between 5 and 37. Expansions from above 50 to several thousand CTG repeats result in myotonic dystrophy.

One aspect of the present invention thus relates to a non human animal model expressing at least one phenotype associated with myotonic dystrophy due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human myotonic dystrophy protein kinase gene DMPK gene mRNA and or protein SEQ ID NO 32 SEQ ID NO 33 wherein the number of CTG repeats is at least 40 45 50 or at least 60.

Similarly in one embodiment the non human animal model expressing at least one phenotype associated with myotonic dystrophy due to the introduction of a genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human DMPK gene mRNA and or protein or fragments or parts thereof respectively wherein the number of CTG repeats in the DNA is at least 40 45 50 or at least 60.

The non human animal model for myotonic dystrophy may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for myotonic dystrophy is generated by sperm mediated gene transfer.

According to the present invention a pig model for myotonic dystrophy is in one embodiment produced by sperm mediated gene transfer of the human or porcine DMPK gene mRNA and or protein wherein the number of CTG repeats in the DNA is at least 40 45 50 or at least 60. However the pig model for myotonic dystrophy may also be produced by introducing the human or porcine DMPK gene mRNA and or protein wherein the number of CTG repeats in the DNA is at least 40 45 50 or at least 60 or protein expressed thereof into a target cell.

The non human animal model for myotonic dystrophy in particular the pig model for myotonic dystrophy will typically develop at least one of the symptoms such as generalized weakness and muscular wasting that affects the face and neck difficulty with the feet that spreads to the legs shoulders and hips. Other symptoms include a wasting of the muscles muscular dystrophy opacity of the lens of the eyes cataracts heart conduction defects and myotonia difficulty in relaxing muscles .

One embodiment of the present invention relates to a non human animal model expressing at least one phenotype associated with fragile X syndrome. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with fragile X syndrome.

Fragile X syndrome is caused by repeats in the promoter region of the human FMR1 gene 6 to 52 CGG repeats are normally present 24 25 . Expansions in the range of 55 to 200 repeats result in the pre mutation while the full mutation ranges from 200 to several thousand repeats resulting in fragile X syndrome. Fragile X syndrome may also be by caused by CCG repeats in the TNR of the 5 end of the human FMR2 gene wherein the number of repeats varies from 6 to 35 26 . Expansions containing from 61 to 200 repeats result in the pre mutation and expansions above 200 repeats result in the full mutation and the fragile X syndrome.

One aspect of the present invention thus relates to a non human animal model expressing at least one phenotype associated with Fragile X syndrome due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human FMR1 gene cDNA mRNA and or protein SEQ ID NO 34 SEQ ID NO 35 wherein the number of CGG repeats is at least 55 60 70 or at least 200 and or of the human FMR2 gene cDNA mRNA and or protein SEQ ID NO 36 SEQ ID NO 37 wherein the number of CCG repeats is at least 61 65 70 80 or at least 200 

Similarly one embodiment relates to the non human animal model expressing at least one phenotype associated with fragile X syndrome due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human FMR1 gene cDNA mRNA and or protein wherein the number of CGG repeats is at least 55 60 70 or at least 200 and or of the porcine homolog of the human FMR2 gene cDNA mRNA and or protein wherein the number of CCG repeats is at least 61 65 70 80 or at least 200 

The non human animal model for fragile X syndrome may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell.

In one particular embodiment the non human animal model for fragile X syndrome is generated by sperm mediated gene transfer.

According to the present invention a pig model for fragile X syndrome is in one embodiment produced by sperm mediated gene transfer of the human or the porcine homolog of the human FMR1 gene cDNA mRNA and or protein fragments or part thereof wherein the number of CGG repeats is at least 55 60 70 or at least 200 and or the human or the porcine homolog of the human FMR2 gene cDNA mRNA and or protein fragment or part thereof wherein the number of CCG repeats is at least 61 65 70 80 or at least 200 However the pig model for myotonic dystrophy may also be produced by introducing the human or porcine FMR1 gene cDNA mRNA and or protein fragments or part thereof wherein the number of CGG repeats is at least 55 60 70 or at least 200 and or the human or the porcine homolog of the human FMR2 gene cDNA mRNA and or protein fragment or part thereof wherein the number of CCG repeats is at least 61 65 70 80 or at least 200 or protein expressed thereof into a target cell.

The non human animal model for fragile X syndrome in particular the pig model for fragile X syndrome will typically develop at least one of the symptoms associated with autism. Non limiting examples of symptoms of autism is for example repetitive behaviour and impairment in social interaction.

One embodiment of the present invention relates to a non human animal model expressing at least one phenotype associated with spinocerebellar ataxia. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with spinocerebellar ataxia.

Spinocerebellar ataxia SCA12 is caused by repeats in the within the 5 UTR of the human PPP2R2B gene and the number of CAG repeats normally varies in size from 7 to 28 and in the expanded form from above 65 to 78 TNRs 27 .

Spinocerebellar ataxia SCA 1 is caused by CAG repeats in the ATX1 protein. The human SCA1 TNR region is characterized by the presence of 12 CAG repeats followed by two CAT repeats flanking a CAG triplet 28 . The CAG TNR prone to expand is normally composed of between 6 and 39 repeats and the expanded version consists of 41 to 81 repeats.

Spinocerebellar ataxia SCA 2 is caused by TNR expansions affecting the ATX2 protein. This TNR normally consists of 15 to 30 CAG repeats and the expanded form ranges from 35 to 59 triplets 29 .

Spinocerebellar ataxia SCA 3 is caused by a CAG TNR expansion in the human ataxin 3 gene wherein the presence of above 54 repeats results in ataxia whereas the normal number of CAG repeats varies between 12 and 36 30 32 .

Spinocerebellar ataxia SCA6 is caused by TNR expansion in the CACNA1A voltage dependent calcium channel results in ataxia 33 . The normal number of TNRs is between 4 and 18 and expansions from 21 to 27 TNRs are disease causative.

Spinocerebellar ataxia SCA7 is caused by TNR of the human SCA7 locus in the N terminal end of the ataxin 7 protein which is normally composed of 7 to 35 CAG repeats 34 . Disease causing expansions range from 37 to 200 repeats.

Spinocerebellar ataxia SCA17 is caused by a CAG expansion in the TATA box binding protein TBP gene and results in the SCA17 phenotype resulting in ataxia 35 . The human TNR region is composed of two groups of CAG repeats separated by multiple CAA and CAG triplets. Expansions normally progress from the larger of the two CAG groups. The normal stretch of encoded poly glutamines varies between 29 and 42 whereas poly glutamine stretches from 47 to 55 have been identified in SCA17 patients.

One aspect of the present invention thus relates to a non human animal model expressing at least one phenotype associated with Spinocerebellar ataxia due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human PPP2R2B gene cDNA mRNA and or protein SEQ ID NO 38 SEQ ID NO 39 wherein the number of CAG repeats is at least 65 70 or at least 75 and or of the human ATX1 gene cDNA mRNA and or protein SEQ ID NO 40 SEQ ID NO 41 wherein the number of 12 CAG repeats followed by two CAT repeats flanking a CAG triplet is at least 41 45 50 60 70 or at least 80. The at least one genetic determinant may also be a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human ATX2 gene cDNA mRNA and or protein SEQ ID NO 42 SEQ ID NO 43 wherein the number of CAG repeats is at least 35 40 45 50 or at least 55 and or the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human ataxin 3 gene cDNA mRNA and or protein SEQ ID NO 44 SEQ ID NO 45 wherein the number of CAG repeats is at least 54 and or the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human CACNA1A gene cDNA mRNA and or protein SEQ ID NO 46 SEQ ID NO 47 wherein the number of TNR expansions is at least 21 22 23 24 25 26 or at least 27 or in the range of 21 to 27 repeats and or the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human N terminal end of the ataxin 7 protein encoding gene cDNA mRNA and or protein SEQ ID NO 48 SEQ ID NO 49 wherein the number of CAG expansions is at least 37 45 55 65 or at least 100 and or the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human TATA box binding protein TBP gene cDNA mRNA and or protein SEQ ID NO 50 SEQ ID NO 51 wherein the number of poly glutamine stretches as defined above is in the range of from 47 to 55 repeats.

Similarly one embodiment relates to the non human animal model expressing at least one phenotype associated with spinocerebellar ataxia due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human PPP2R2B gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 65 70 or at least 75 and or of the porcine homolog of the human ATX1 gene cDNA mRNA and or protein wherein the number of 12 CAG repeats followed by two CAT repeats flanking a CAG triplet is at least 41 45 50 60 70 or at least 80. The at least one genetic determinant may also be a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human ATX2 gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 35 40 45 50 or at least 55 and or the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human ataxin 3 gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 54 and or the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human CACNA1A gene cDNA mRNA and or protein wherein the number of TNR expansions is at least 21 22 23 24 25 26 or at least 27 or in the range of 21 to 27 repeats and or the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human N terminal end of the ataxin 7 protein encoding gene cDNA mRNA and or protein wherein the number of CAG expansions is at least 37 45 55 65 or at least 100 and or the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human TATA box binding protein TBP gene cDNA mRNA and or protein wherein the number of poly glutamine stretches as defined above is in the range of from 47 to 55 repeats.

The non human animal model for spinocerebellar ataxia may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for spinocerebellar ataxia is generated by sperm mediated gene transfer.

According to the present invention a pig model for spinocerebellar ataxia is in one embodiment produced by sperm mediated gene transfer of the human or the porcine homolog of the human PPP2R2B gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 65 70 or at least 75 and or the porcine homolog of the human or the human ATX1 gene cDNA mRNA and or protein wherein the number of 12 CAG repeats followed by two CAT repeats flanking a CAG triplet is at least 41 45 50 60 70 or at least 80 and or the human or the porcine homolog of the human ATX2 gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 35 40 45 50 or at least 55 and or the human or the porcine homolog of the human ataxin 3 gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 54 and or the human or the porcine homolog of the human CACNA1A gene cDNA mRNA and or protein wherein the number of TNR expansions is at least 21 22 23 24 25 26 or at least 27 or in the range of 21 to 27 repeats and or the human or the porcine homolog of the human N terminal end of the ataxin 7 protein encoding gene cDNA mRNA and or protein wherein the number of CAG expansions is at least 37 45 55 65 or at least 100 and or the human or porcine homolog of the human TATA box binding protein TBP gene cDNA mRNA and or wherein the number of poly glutamine stretches as defined above is in the range of from 47 to 55 repeats.

However the pig model for spinocerebellar ataxia may also be produced by introducing the human or porcine homolog of PPP2R2B ATX1 ATX2 ataxin 3 CACNA1A ataxin 7 and or TATA box binding protein TBP in any combination.

The non human animal model for spinocerebellar ataxia in particular the pig model for spinocerebellar ataxia will typically develop at least one of the symptoms such as atrophy of the cerebellum which can be seen by magnetic resonance imaging and or poor coordination of movement.

One embodiment of the present invention relates to a non human animal model expressing at least one phenotype associated with DRPLA. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with DRPLA.

DRPLA is caused by CAG expansions within the human atrophin 1 gene results in dentatorubral pallidoluysian atrophy DRPLA 36 . The normal range of repetitive CAG repeats is from 3 to 25 and in patients with DRPLA allele sizes have expanded to 49 to 88 CAG repeats. The most common natural occurring human allele encodes a stretch of 17 poly glutamines.

Thus one aspect of the present invention thus relates to a non human animal model expressing at least one phenotype associated with DRPLA due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human atrophin 1 cDNA mRNA and or protein SEQ ID NO 52 SEQ ID NO 53 wherein the number of CAG repeats is at least 49 55 60 70 or at least 80 repeats.

Similarly one embodiment relates to the non human animal model expressing at least one phenotype associated with DRPLA due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human atrophin 1 gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 49 55 60 70 or at least 80 repeats.

The non human animal model for DRPLA may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for DRPLA is generated by sperm mediated gene transfer.

According to the present invention a pig model for DRPLA is in one embodiment produced by sperm mediated gene transfer of the human or the porcine homolog of the human atrophin 1 gene cDNA mRNA and or protein fragments or part thereof wherein the number of CAG repeats is at least 49 55 60 70 or at least 80 repeats.

However the pig model for DRPLA may also be produced by introducing the human or porcine homolog of the human atrophin 1 gene cDNA mRNA and or protein fragments or part thereof wherein the number of CAG repeats is at least 49 55 60 70 or at least 80 repeats or protein expressed thereof into a target cell.

The non human animal model for DRPLA in particular the pig model for ALS will typically develop at least one of the symptoms epileptic seizures myoclonus ataxia and dementia.

One embodiment of the present invention relates to a non human animal model expressing at least one phenotype associated with SBMA. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with SBMA.

SBMA is caused by CAG repeat expansions in exon 1 of the androgen receptor AR gene on the X chromosome results in spinal and bulbar muscular atrophy Kennedy s disease 37 . The normal length of the human CAG TNR is between 11 and 33 CAG copies and in diseased individuals the expansion ranges from 38 to 62.

Thus one aspect of the present invention thus relates to a non human animal model expressing at least one phenotype associated with SBMA due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human AR gene cDNA mRNA and or protein SEQ ID NO 54 SEQ ID NO 55 wherein the number of CAG repeats is at least 38 45 50 55 or at least 60 or in the range of 38 to 62 repeats.

Similarly one embodiment relates to the non human animal model expressing at least one phenotype associated with SBMA due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human AR gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 38 45 50 55 or at least 60 or in the range of 38 to 62 repeats.

The non human animal model for SBMA may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for SBMA is generated by sperm mediated gene transfer.

According to the present invention a pig model for SBMA is in one embodiment produced by sperm mediated gene transfer of the human or the porcine homolog of the human AR gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 38 45 50 55 or at least 60 or in the range of 38 to 62 repeats.

However the pig model for SBMA may also be produced by introducing the human or porcine homolog of the human AR gene cDNA mRNA and or wherein the number of CAG repeats is at least 38 45 50 55 or at least 60 or in the range of 38 to 62 repeats or protein expressed thereof into a target cell.

The non human animal model for SBMA in particular the pig model for SBMA will typically develop at least one of the symptoms uncontrollable twitching fasciculations followed by weakness and wasting of the muscles becomes apparent sometime after the age of fifteen. The muscles of the face lips tongue mouth throat vocal chords trunk and limbs may be affected.

One embodiment of the present invention relates to a non human animal model expressing at least one phenotype associated with HD. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with HD.

 Huntington s disease also known as Huntington chorea is used herein to refer to any inherited condition characterized by abnormal and or uncontrolled body movements mental and emotional problems and loss of thinking ability cognition .

Adult onset Huntington disease the most common form of this disorder usually begins in middle age. Signs and symptoms can include irritability depression small involuntary movements poor coordination and trouble learning new information or making decisions. As the disease progresses involuntary jerking movements chorea become more pronounced. Affected individuals may have trouble walking speaking and swallowing. People with the disorder also typically experience changes in personality and a decline in thinking and reasoning abilities. Individuals with this form of Huntington disease generally survive about 15 to 25 years after onset.

There is also an early onset form of Huntington disease that begins in childhood or adolescence. Some of the clinical features of this disease differ from those of the adult onset form. Signs and symptoms can include slowness clumsiness rigidity loss of developmental milestones such as motor skills slow speech and drooling. Seizures occur in 30 percent to 50 percent of individuals with this condition. The course of early onset Huntington disease may be shorter than adult onset Huntington disease affected individuals generally survive 10 to 15 years after onset.

Huntington s disease in humans is linked to the Huntingtin gene HD gene accession number NM002111 . The function of the corresponding protein is not yet known but it likely plays an important role in nerve cells. The disease causing mutation in Huntington s disease is an extension of a CAG repeat to a length above 35 CAG units. The number of repeats can to a certain extent be correlated with disease onset.

The expanded repeat leads to the production of a huntingtin protein that contains a long stretch of the amino acid glutamine. The elongated protein disrupts the normal function of nerve cells in certain parts of the brain and ultimately leads to the death of those cells. The dysfunction and loss of nerve cells cause the signs and symptoms of Huntington disease.

Thus one aspect of the present invention thus relates to a non human animal model expressing at least one phenotype associated with HD due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human Huntingtin gene cDNA mRNA and or protein SEQ ID NO 56 SEQ ID NO 57 wherein the number of CAG repeats is at least 35 45 50 55 or at least 60.

Similarly one embodiment relates to the non human animal model expressing at least one phenotype associated with HD due to the introduction of at least one genetic determinant wherein the at least one genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine homolog of the human Huntingtin gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 35 45 50 55 or at least 60.

The non human animal model for HD may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for HD is generated by sperm mediated gene transfer.

According to the present invention a pig model for HD is in one embodiment produced by sperm mediated gene transfer of the human or the porcine homolog of the human Huntingtin gene cDNA mRNA and or protein wherein the number of CAG repeats is at least 35 45 50 55 or at least 60.

However the pig model for HD may also be produced by introducing the human or porcine homolog of the human huntingtin gene cDNA mRNA and or wherein the number of CAG repeats is at least 35 45 50 55 or at least 60 or protein expressed thereof into a target cell.

The non human animal model for HD in particular the pig model for HD will typically develop at least one of the symptoms described above such as slowness clumsiness rigidity loss of developmental milestones such as motor skills slow speech and drooling or seizures.

In one embodiment of the present invention the non human animal model expresses at least one phenotype associated with dyschondroplasia. In a particular aspect the non human animal model is a pig model expressing at least one phenotype associated with dyschondroplasia.

Chondrodysplasias are a group of disorders affecting the skeleton and are often associated with proteins of the collagen superfamily constituting at least 19 different types of collagen these being major components of cartilages throughout the mammalian organism implicating roles in the processes of calcification and ossification 38 . One third of the collagens types I II III V and IX are denoted fibrillar collagens due to their tissual status as long highly organised fibers. The remaining two thirds of the collagens are nonfibrillar and are further divided into two groups the fibril associated with interrupted helices and the network forming collagens 39 . Collagen X belongs to the group of network forming collagens and has been reported to be expressed mainly in the hypertrophic region of growth plate cartilage 40 and has only rarely been detected in calcifying region of articular cartilage 41 .

Type X collagen has been suggested to be structurally important in the extracellular matrix by offering the molecular milieu essential for endochondral bone formation 40 . Furthermore it has been shown that type X collagen exists in ostechondrotic and osteoarthritic porcine articular cartilage and it may be a product of the cell population trying to repair the breakdown 41 . However in osteochondrosis the endochondral ossification is impaired regardless of the unaltered collagen X levels in the growth plate and the increase of said collagen in articular cartilage. This could indicate that type X collagen alone is not able to cause ossification of cartilage 41 . Moreover it has previously been shown in a naturally occurring porcine model that a mutation in the porcine noncollagenous domain 1 NC1 of collagen X causes a phenotype similar to the human dwarfism Schmid metaphyseal chondrodysplasia SCMD 42 . Said cartilage disorder is an autosomal dominant disease which characteristics are short stature coxa vara and a waddling gait and histopathological examinations show an extremely irregular organisation of the growth plate in long bones 43 44 .

Transgenic animals are important tools as model organisms in basic research as well as in applied scientific areas and they have now been used for several years to study for instance gene function and human diseases. In relation to transgenesis collagen X offers a unique opportunity regarding the detection of transgene expression since wild type collagen X is almost selectively expressed in chondrocytes making the detection of transgene expression in other tissues uncomplicated and hence easy to ascribe to the transgenic procedure.

One aspect of the present invention thus relates to a non human animal model expressing at least one phenotype associated with dyschondroplasia due to the introduction of at least one genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the human COL10 A1 gene mRNA and or protein SEQ ID NO 58 SEQ ID NO 59 being present in at least a transient manner.

Similarly in one embodiment the non human animal model expressing at least one phenotype associated with dyschondroplasia due to the introduction of a genetic determinant wherein the genetic determinant is a gene or DNA or fragment thereof and or RNA or fragment thereof and or protein or part thereof of the porcine COL10A1 gene mRNA and or protein corresponding to SEQ ID NO 60 SEQ ID NO 61 or fragments or parts thereof being present in at least a transient manner.

The non human animal model for dyschondroplasia may be generated by introduction of said genetic determinant into a target cell by any method available to the skilled person for example by injection into the target cell or by virus mediated transfer or any method suitable as known by the skilled person.

In one particular embodiment the non human animal model for dyschondroplasia is generated by sperm mediated gene transfer.

According to the present invention a pig model for dyschondroplasia is in one embodiment produced by sperm mediated gene transfer of the constitutively expressed porcine COL10A1 of porcine or human origin as described elsewhere herein. However the pig model for dyschondroplasia may also be produced by introducing the COL10A1 DNA or protein expressed thereof into a target cell.

The non human animal model for dyschondroplasia in particular the pig model for dyschondroplasia will typically develop at least one of the symptoms described above such as disorders involving tubular bones and characterized by a neoplasmlike proliferation of cartilage in the metaphyses that cause distorted growth in length or pathological fractures.

The present invention relates to a non human animal serving as a disease model for autosomal dominant disorders for example neurodegenerative diseases such as protein conformation diseases such as listed elsewhere herein.

In one embodiment the non human animal model may be any model with the proviso that the animal is not a rodent such as rat mouse or hamster. The non human animal is selected from the group consisting of ape monkey cattle pig sheep goat horse donkey.

In one embodiment the pig presenting the pig model is a wild pig. In another embodiment the pig is the domestic pig such as . In yet another embodiment the invention relates to mini pigs as well as to inbred pigs. The pig can be selected e.g. from the group consisting of Landrace Yorkshire Hampshire Duroc Chinese Meishan Berkshire and Pietrain such as the group consisting of Landrace Yorkshire Hampshire and Duroc for example the group consisting of Landrace Duroc and Chinese Meishan such as the group consisting of Berkshire Pietrain Landrace and Chinese Meishan for example the group consisting of Landrace and Chinese Meishan. In one embodiment the pig is not a mini pig.

In another embodiment of the present invention the pig is a mini pig and the mini pig is preferably selected from the group consisting of Goettingen Yucatan Bama Xiang Zhu Wuzhishan and Xi Shuang Banna.

One aspect of the present invention relates to a non human animal model produced by sperm mediated gene transfer. The method of sperm mediated gene transfer of the present invention for the production of a non human animal model for studying a hereditary autosomal dominant disease and or for the production of a pig model for studying amyotrophic lateral sclerosis Alzheimer s disease Parkinson s disease diseases associated with trinucleotide repeats Huntington s chorea and or dyschondroplasia comprises the steps of i providing semen from a male non human animal ii providing a genetic determinant capable of establishing said hereditary disease when the genetic determinant is expressed in said non human animal model iii contacting said semen and said genetic determinant iv fertilising an oocyte from a female non human animal with the semen and the genetic determinant and v incubating said fertilised oocyte under conditions allowing said fertilised oocyte to develop into said non human animal model.

The semen for the method is provided from a male non human animal in particular a boar. The selection of the sperm donor boars is crucial for the outcome of the procedure. The boars of choice are selected so that the initial sperm motility is 90 . Preferably the semen is fresh and collected in sterile 10 mL tubes and transported undiluted at a temperature not below 15 C. as this will cause damage to the sperm cells. The quality of the sperm cell and thus the efficiency of the sperm mediated gene transfer procedure is affected by numerous factors such as season of year collection frequency breed and age of the donor. In order to choose the correct donor cells the sperm cells from the different boars are examined under a light microscope. The sperm cells originating from the boar having the highest sperm cell motility after the washing procedure are chosen. Next the sperm cells from the boar of choice are counted. It is important for the present invention that the sperm cell motility is maintained following the removal of seminal fluid.

In one embodiment the boar has abstained for 1 to 10 days 2 8 days 1 2 days or in a particular embodiment of the invention the donor boar has abstained for 2 days prior to collecting the semen to be employed in the procedure.

The mechanism of internalization of foreign DNA involves specific proteins capable of binding DNA in a CD4 like manner to sperm heads. The proteins of 30 35 kDa have been identified in a variety of species such as mice cattle pig and humans. However under normal conditions the seminal fluid strongly protects the sperm cells from foreign DNA by antagonizing the binding of the DNA to be internalized. The main antagonist is inhibitory factor 1 IF 1 . Accordingly the antagonist has to be removed or neutralised before the sperm is used in sperm mediated gene transfer procedures. Therefore in one embodiment of the present invention the seminal fluid is removed from the spermatozoa for example by washing.

Ejaculated spermatozoa will under normal conditions capacitate which means that the spermatozoa undergo physiological changes rendering the cells able to fertilise. Therefore in one embodiment of the present invention the sperm mediated gene transfer method the initiation of the sperm DNA interaction should be started shortly after removal of the seminal fluid for example during the washing step of the procedure . In another embodiment in order to facilitate capacitation and correct sperm DNA incubation time the applied buffer is calcium free as calcium under normal conditions promotes capacitation. Additionally the absence of calcium prevents endonucleases from acting on the foreign DNA. Thus in a particular embodiment of the invention the sperm is washed in a buffer devoid of calcium. In a further particular embodiment the washing buffer comprises the following components are 56.1 g Glucose 3.5 g EDTA 2H0 3.5 g Sodium citrate 1.1 g Sodium bicarbonate for 1 liter of buffer dissolved in water and the solution is sterilized through a filter. Before the buffer is added to the sperm cells 6 mg ml BSA Bovine Serum Albumine Fraction V Sigma is added.

One preferred washing procedure according to the present invention is accomplished as fast as possible as follows 5 mL sperm is transferred to a 50 mL tube and 5 mL washing buffer preheated to 37 C. is added mix by inverting the tubes. The solution is incubated for 5 min at room temperature approximately 22 C. and 40 mL buffer room temperature is added. Upon centrifugation at 800 g for 10 min at 25 C. the supernatant is removed and the pellet resuspended in 50 mL buffer room temperature . The mixture is subjected to centrifugation at 800 g for 10 min at 17 C. and the supernatant is removed. The pellet is carefully resuspended in the remaining buffer in the tube.

When the genetic determinant has been provided the semen is contacted with the genetic determinant. Sperm cells are diluted in buffer as described above for the washing of sperm cells. In one embodiment the buffer temperature is in the range of 15 C. to 25 C. and in particular the temperature is 17 C. The genetic determinant in the form of DNA is added to the diluted sperm cells. The concentration of the genetic determinant is in the range of 0.1 g to 1 g per 10sperm cells. In a particular embodiment the genetic determinant is linearised DNA added in a concentration of 0.4 g linearised DNA 10sperm cells.

The non human animal model and in particular the pig models of the present invention may be produced by methods other than sperm mediated gene transfer for example by pronuclear microinjection somatic cell nuclear transfer or retrovirus mediated gene transfer.

A further aspect of the present invention pertains to a non human sperm cell comprising at least one genetic determinant exerting at least one dominant phenotype for at least one hereditary disease when expressed in a non human animal host organism. The non human sperm cell may originate from any of the animals listed elsewhere herein. As described in detail elsewhere herein the genetic determinant is of mammalian origin for example of human and or porcine origin. The non human sperm cell may be a non human sperm cell exerting an autosomal dominant phenotype for a hereditary disease such as any of the diseases according to the present invention. The non human host organism is any of the animals presented herein and in particular a pig.

The non human sperm cell according to the present invention may be produced by a method comprising the steps of a providing a non human sperm cell b providing at least one genetic determinant exerting a dominant phenotype for a hereditary disease when expressed in a non human animal host organism c contacting said non human sperm cell and said at least one genetic determinant wherein said contacting results in the uptake of the genetic determinant into the non human sperm cell.

A further aspect of the invention relates to a composition comprising a non human sperm cell in combination with at least one genetic determinant exerting at least one dominant phenotype for at least one hereditary disease when expressed in a non human animal host organism. In preferred embodiments the genetic determinant is of human and or porcine origin.

The present invention also discloses a method for fertilising an oocyte by sperm mediated gene transfer. The method comprises the steps of providing the non human sperm cell as carrying the at least one genetic determinant for a phenotype associated with autosomal dominant diseases as defined herein and introducing the non human sperm cell into the oocyte to be fertilised. Consequently another aspect of the invention relates to a method for fertilising an oocyte by sperm mediated gene transfer wherein the method comprises the steps of providing the composition as described herein and introducing the composition into the oocyte to be fertilised.

Therefore the present invention also pertains to an embryo obtained by fertilising an oocyte with the non human sperm cell comprising at least one genetic determinant exerting at least one dominant phenotype for at least one hereditary disease of the present invention when expressed in a non human animal host organism. Similarly an embryo obtained by fertilising an oocyte with the composition as disclosed herein is within the scope of the present invention. Furthermore as an embryo has been established the present invention offers a method for the cultivation and development of the embryo comprising the step of cultivating the embryo under conditions allowing the embryo to develop into a non human animal offspring expressing said genetic determinant and exerting a dominant phenotype for a hereditary disease.

The presence of a non human animal model of autosomal dominant diseases provides the opportunity of evaluating whether a given pharmaceutical composition compound treatment and or drug has an effect on the phenotype of the given non human animal. Therefore it is within the scope of the present invention to provide for a method for evaluating the response of a therapeutical treatment of a hereditary disease said method comprising the steps of a providing the non human animal model according to the invention b treating said non human animal with at least one pharmaceutical composition exerting an effect on said at least one phenotype and c evaluating the effect observed. Additionally the method also allows for a further step of advising on medical treatment of for example a human being suffering from an autosomal dominant disease such as a neurodegenarative diseases protein conformation diseases such as ALS Alzheimer s disease HD PD trinucleotide repeat associated diseases but also dyschondroplasia based on the effects observed during the method of evaluation.

In addition the availability of a non human animal model expressing a particular phenotype associated with autosomal dominant diseases offers the ability of providing a method for screening the efficacy of a pharmaceutical composition wherein the method comprises the steps of a providing the non human animal model of the present invention b expressing in said animal model said at least one genetic determinant and exerting said dominant phenotype for said hereditary disease c administering to said non human animal the pharmaceutical composition the efficacy of which is to be evaluated and d evaluating the effect if any of the pharmaceutical composition on the phenotype exerted by the genetic determinant when expressed in the non human model.

Furthermore the present invention also relates to a method for treatment of a human being suffering from an autosomal dominant disease wherein the method comprises the initial steps of a providing the non human animal model of the present invention b expressing in said animal model said genetic determinant and exerting said dominant phenotype for said hereditary disease c treating said non human animal with a pharmaceutical composition exerting an effect on said phenotype d evaluating the effect observed and e treating said human being suffering from said hereditary disease based on the effects observed in the non human animal model.

Moreover a method for linking a genetic determinant with the occurrence of a hereditary disease in a human being is also disclosed said method comprising the steps of a cultivating and developing an embryo obtained by fertilising an oocyte with a non human sperm cell and a genetic determinant potentially constituting an autosomal dominant disease gene for example a porcine gene exerting a dominant phenotype for a disease such as a neurodegenerative disease when expressed in a non human animal host organism wherein said cultivation and development result in the generation of an non human animal offspring and b observing whether said genetic determinant confers said autosomal dominant disease in said non human animal offspring.

In order to establish transgenic pigs which could serve as potential animal models for the human neurodegenerative disease ALS a mutation Gly93Arg was introduced into the gene encoding porcine SOD1 by means of site directed mutagenesis. The choice of mutation was based on protein structural speculations since the crystal structure of human SOD1 reveals an extremely condensed structure showing that a substitution of the small relatively flexible glycine at position with the large charged arginine is likely to cause severe alteration in the protein. Furthermore several different substitutions at this position cause ALS in humans including the G93R mutation 45 46 . In order to impede possible truncation of important elements in the DNA construct following the SMGT procedure the DNA fragment containing the porcine SOD1 cDNA contains additional nucleotides 5 and 3 prime to the CMV promoter and SVpolyA fragment. Totally as shown in the fragment used to make transgenic animals constitutes approximately 2100 bp.

Various porcine tissues were dissected from slaughtered pigs Duroc boars and Landrace Yorkshire sows D LY and immediately frozen in liquid nitrogen and stored at 80 C. 100 mg of the tissue of choice was used for RNA isolation. RNA was isolated using the Nucleospin Midi Kit from Macherey Nagel.

cDNA synthesis was accomplished by mixing 1 g of total RNA with 1 L of oligo dT 12 18 500 g mL and DEPC treated H20 to a final volume of 12 L. The mixture was incubated at 70 C. for 10 min after which 4 L of 5 first strand buffer 2 L of 0.1 mM DTT 1 L of 10 mM dNTP mix and 1 L 200 U L of Superscript II Invitrogen was added and the sample was further incubated at 42 C. for 1 hour followed by an inactivation step at 70 C. for 15 min.

Based on homology search between the human SOD1 gene and an in house porcine EST database 2 primers SOD1 CDF and SOD1 CDR were designed for amplification of the cDNA for the porcine SOD1 gene. The PCR reaction was performed in a total volume of 25 L consisting of 2.5 L of 10 reaction buffer 4 L of dNTP 2.5 mM 1 L of both forward and reverse primer 10 pmol each 1 L 1 U L Dynazyme polymerase Finnzymes 2 L cDNA template and 13.5 L H20.

The touchdown PCR reaction was performed in a GeneAmp PCR System 9700 Applied Biosystems under the following conditions Initial denaturation at 95 C. for 3 min denaturation at 95 C. for 30 sec touchdown from 63 C. to 58 C. with a decrement of 0.5 C. for 30 C. followed by 1 min of elongation at 72 C. per cycle. Furthermore 35 cycles of 30 sec denaturation at 95 C. 30 sec of annealing at 58 C. and 1 min of elongation at 72 C. was included together with a final elongation step at 72 C. for 7 min.

Amplicons were cloned into the pCR 2.1 TOPO vector Invitrogen CA using manufactures recommendations and applying standard procedures clones were subsequently sequenced to ensure that they contained the SOD1 amplicon.

Cycle sequencing reactions were accomplished in a GeneAmp PCR System 9700 Applied Biosystems where an initial denaturation step at 95 C. for 2 min 99 cycles of 10 sec denaturation and 4 min elongation at 60 C. was applied to the sample mixture consisting of 1.5 L of Big Dye Terminator mix version 3.1 1 L of primer 5 pmol 2 L of a 5 reaction buffer 2 L of the purified PCR product and 3.5 L H20. Sequencing product were precipitated with 2.5 volumes of ethanol and 1 10 volume of 3 M NaAc air dried and resuspended in 10 L formamide sequencing grade . The samples were run on a 3730xl DNA Analyzer Applied Biosystems .

In order to introduce the Gly93Arg mutation into SOD1 site directed mutagenesis was performed using the QuickChange XL Site Directed Mutagenesis Kit Stratagene and accomplished in accordance with the manufacturer s recommendations as described herein. The PCR reaction was performed in a total volume of 50 L consisting of 5 L of 10 reaction buffer 2 L of both forward and reverse primer 10 pmol each 1 L dNTP mix 3 L QuickSolution 1 L PfuTurbo DNA polymerase 2.5 U L 1 L TOPO SOD1 template 10 ng and 35 L H20.

The PCR reaction was accomplished in a GeneAmp PCR System 9700 Applied Biosystems under the following conditions Initial denaturation at 95 C. for 1 min 18 cycles of 50 sec denaturation at 95 C. 50 sec of annealing at 60 C. and 6 min of elongation at 68 C. and a final elongation step at 68 C. for 7 min.

Subsequently 1 L of Dpn I 10 U L was added to the sample mixture in order to digest the nonmutated parental DNA template. The reaction was incubated for 1 hour at 37 C. After digestion of the parental DNA template XL10 Gold Ultracompetent Cells Invitrogen CA were transformed with the Dpn I treated DNA in the following manner XL10 Gold Ultracompetent Cells were thawn on ice and 45 L were aliquoted to a prechilled Eppendorf tube and 2 L of mercaptoethanol was added followed by a gentle swirling of the tube. The tube was now left on ice for 10 min. After incubation 2 L of the Dpn I treated DNA was added the sample was swirled and incubated on ice for another 30 min and then exposed to a heat pulse of 42 C. for 30 sec. Subsequently the sample was put on ice for 2 min followed by an addition of 0.5 mL 42 C. NZY broth. The mix was plated onto LB amp plates and incubated overnight at 37 C. To ensure that the mutation of interest was integrated in the porcine SOD1 gene 16 colonies were picked and grown overnight in liquid LB Amp media and plasmids were purified using the QIAprep Spin Miniprep kit Qiagen according to manufactures recommendations. Cycle sequencing reactions were accomplished in a GeneAmp PCR System 9700 Applied Biosystems where an initial denaturation step at 95 C. for 2 min 99 cycles of 10 sec denaturation and 4 min elongation at 60 C. was applied to the sample mixture consisting of 1.5 L of Big Dye Terminator mix version 3.1 1 L of either T7 or SP6 primer 5 pmol 2 L of a 5 reaction buffer 1 L of the purified plasmid and 4.5 L H20. Sequencing product were precipitated with 2.5 volumes of ethanol and 1 10 volume of 3 M NaAc air dried and resuspended in 10 L formamide sequencing grade . The samples were run on a 3730xl DNA Analyzer Applied Biosystems . Sequences were checked and a plasmid containing the mutation was chosen as template for the subsequent procedures. The sequence of the mutated porcine SOD1 cDNA is shown in .

To facilitate a continuous high expression of the transgene of interest the gene was cloned into the phCMV1 vector Gene Therapy Systems . For release of the mutated SOD1 DNA 6 L plasmid DNA was digested with 1.5 L EcoRI 20 U L in a total volume of 30 L in addition of 3 L EcoRI 10 reaction buffer. The reaction was incubated at 37 C. for 90 min and run on a 1 Seakem GTG agarose gel. The correctly sized band was isolated from the agarose gel employing a QIAEX II Gel Extraction Kit Qiagen and dissolved in 30 L HO. Likewise phCMV1 was EcoRI digested and isolated from a 0.8 Seakem GTG agarose gel and dissolved in 30 L HO. Furthermore in order to avoid self ligation of the vector 6 L of the digested phCMV1 vector was dephosphorylated in a total volume of 25 L applying 1.5 L CIP 10 U L and 2.5 L 10 CIP reaction buffer. The sample was incubated 60 min at 37 C. Enzyme inactivation occurred at 80 C. for 15 min.

Ligation of mutagenised SOD1 into the EcoRI digested and dephosphorylated phCMV1 was performed in a total volume of 15 L in the addition of 3 L dephosphorylated phCMV1 8 L EcoRI linked mutagenised SOD1 1.5 L T4 DNA ligase 400 U L 1.5 L 10 T4 DNA ligase buffer and 1 L HO. The reaction was incubated at 16 C. overnight. XL10 Gold Ultracompetent Cells were thawn on ice and 45 L were aliquoted to a prechilled ependorf tube and 2 L of mercaptoethanol was added followed by a gentle swirling of the tube. The sample was now left on ice for 10 min. After incubation 3 L of the ligation mix was added the sample was swirled and incubated on ice for another 30 min and then exposed to a heat pulse of 42 C. for 30 sec. Subsequently the sample was put on ice for 2 min followed by an addition of 0.5 mL 42 C. NZY broth. The mix was plated onto LB amp plates and incubated overnight at 37 C.

To ensure that the insert had integrated correctly into the phCMV1 vector 16 colonies were picked and grown overnight in liquid LB Amp media and plasmids were purified using the QIAprep Spin Miniprep kit Qiagen in accordance with manufactures recommendations. Cycle sequencing reactions were accomplished in a GeneAmp PCR System 9700 Applied Biosystems where an initial denaturation step at 95 C. for 2 min 99 cycles of 10 sec denaturation and 4 min elongation at 60 C. was applied to the sample mixture consisting of 1.5 L of Big Dye Terminator mix version 3.1 1 L of either T7 or SP6 primer 5 pmol 2 L of a 5 reaction buffer 1 L of the purified plasmid and 4.5 L HO Sequencing product was precipitated with 2.5 volumes of ethanol and 1 10 volume of 3 M NaAc air dried and resuspended in 10 L formamide sequencing grade . The samples were run on a 3730xl DNA Analyzer Applied Biosystems . Sequences were checked and a plasmid containing the SOD1 construct in the correct direction was chosen.

In order to create DNA for incubation of sperm cells both PCR and large scale plasmid preparations have been employed.

The PCR reaction was performed in a GeneAmp PCR System 9700 Applied Biosystems in a final volume of 25 L consisting of 5 L 5 Phusion HF buffer 2 L dNTP 2.5 mM each 0.63 L forward and reverse primer 5 pmol 0.1 L Phusion DNA Polymerase 2 U L 1 L SOD1 phCMV1 template and 15.6 L HO. The PCR reaction consisted of an initial denaturation at 98 C. for 30 sec followed by 30 cycles of denaturation for 10 sec at 98 C. annealing at 74 C. for 30 sec and elongation for 95 sec at 72 C. followed by a final elongation step at 72 C. for 7 min.

The following primers were used to amplify the mutagenised SOD1 construct plus the flanking f r var termen matching CMV promoter intron sequence and SVpolyA generating a fragment of 1643 bp the mutagenised SOD1 fragment generating a fragment of approximately 2100 bp 

In order to purify the generated PCR product a High Pure PCR Product Purification Kit Roche was applied. The suppliers instructions were followed throughout the purification procedure. The PCR purified fragments were sequenced to check for errors in the sequence as described below.

Large scale plasmid preparation was accomplished from liter cell cultures. Purification of plasmids was performed using a Plasmid Mega Kit Qiagen . In order to linearise and release the desired fragment from the vector the vector was digested with BssS I and Dra III in the following way 1.5 L BssSI 4 U L 1.5 L DraIII 20 U L 3 L 10 BSA 3 L 10 Ne buffer 3 and 2 L plasmid DNA was added to 19 L HO to yield a total volume of 30 L and was left overnight at 37 C. The digested fragments were separated on a 0.8 GTG Seakem agarose gel and the correctly sized band were isolated and extracted from the gel using QIAquick Gel Extraction Kit Protocol Qiagen according to manufactures recommendations.

Both the PCR purified fragments and the plasmid prepared fragments were sequenced to check for errors in the sequence. Cycle sequencing reactions were accomplished in a GeneAmp PCR System 9700 Applied Biosystems where an initial denaturation step at 95 C. for 2 min 99 cycles of 10 sec denaturation and 4 min elongation at 60 C. was applied to the sample mixture consisting of 1.5 L of Big Dye Terminator mix version 3.1 1 L of either T7 or SP6 primer 5 pmol 2 L of a 5 reaction buffer 1 L of the purified plasmid and 4.5 L HO. Sequencing product were precipitated with 2.5 volumes of ethanol and 1 10 volume of 3 M NaAc air dried and resuspended in 10 L formamide sequencing grade . The samples were run on a 3730xl DNA Analyzer Applied Biosystems .

The selection of the sperm donor boars are crucial for the outcome of the procedure. A boar station Hatting KS Viborg have therefore been contacted and the boars of choice are selected so that the initial sperm motility is 90 . The sperm is collected in sterile 10 mL tubes and transported undiluted at a temperature not below 15 C. as this will cause damage to the sperm cells.

First and second semen ejaculate were collected from 8 different boars yielding 16 semen fractions in total. All fractions of spermatozoa had an initial motility of 90 prior to the washing procedure. Seminal fluid was quickly removed by centrifugation and washing the sperm in Fertilization Buffer FB consisting of 56.1 g Glucose 3.5 g EDTA 2HO 3.5 g Sodium Citrate 2HO and 1.1 g sodium bicarbonate dissolved in 1 liter of sterilized water. Furthermore 6 mg ml BSA Fraction V Sigma was added. Briefly 5 mL of FB BSA prewarmed to 37 C. was added to 5 mL of undiluted semen mixed by inverting the tube and left for 5 minutes at room temperature approximately 22 C. . Next FB BSA at room temperature was added to 50 ml and centrifuged for 10 minutes at room temperature at 500 g or alternatively at 800 g for 10 min at 25 C. The supernatant was removed and semen was resuspended in 50 mL FB BSA at room temperature and further centrifuged at 500 g at 17 C. after which the supernatant was removed again and the spermatozoa was resuspended in 15 mL of FB BSA. Next in order to choose the optimal donor cells the spermatozoa from the different boars and the two separate ejaculates were quickly examined under a light microscope. The sperm cells originating from the two boars having the highest sperm cell motility after the washing procedure were chosen as vehicles for the subsequent transgenic procedures. Furthermore the spermatozoa were counted.

1 10sperm cells washed spermatozoa from each of the two chosen donor boars were incubated for 100 minutes at 17 C. with the linearized SOD1 DNA fragment in a concentration of 0.4 g DNA 106 spermatozoa in a suspension of 120 mL FB BSA. Containers were inverted every 20 minutes to prevent sedimentation of spermatozoa. Finally the mixture was incubated 10 minutes at room temperature and employed in artificial insemination of two sows in natural heat.

Two recipient sows Danish Landrace Yorkshire at approximately 140 kg were selected due to their natural heating period and used for artificial insemination 1 10DNA treated sperm spermatozoa sow meeting standard insemination procedures. Insemination was accomplished in the local stable areas at DIAS. Semen was collected from trained Danish Landrace boars that have abstained for 2 days. Semen was treated according to aforementioned procedures. Both sows were examined for pregnancy 24 and 42 days after insemination showing that only one of the sows was pregnant. After ended gestation period 2 boars and 5 sow piglets were born naturally. Animal care and experimental procedures met local national and European Union Guidelines.

After 115 days 20 Jun. 2005 7 normal looking piglets were born 2 of these were boar piglets and 5 were sow piglets. Blood samples were collected from the piglets in 6 mL EDTA blood collection tubes. Furthermore blood from a wild type animal was collected as well and handled together with the 7 aforementioned animals. DNA was purified according to standard blood purification procedures in special clean laboratories in order to avoid any possible contamination.

The PCR reaction was performed in a total volume of 10 L consisting of 1 L 10 MgCl2 free reaction buffer 0.4 L 50 mM MgCl2 1 L of both forward and reverse primer 10 pmol each 0.5 L dNTP mix 0.5 L DyNazyme EXT DNA polymerase 1 U L 0.5 L DNA template 50 ng and 5.1 L HO.

The touchdown PCR reaction was accomplished in a GeneAmp PCR System 9700 Applied Biosystems under the following conditions Initial denaturation at 95 C. for 3 min denaturation at 95 C. for 30 sec touchdown from 62 C. to 57 C. with a decrement of 0.5 C. for 20 sec followed by 1 min of elongation at 72 C. pr cycle. Furthermore 35 cycles of 30 sec denaturation at 95 C. 20 sec of annealing at 57 C. and 1 min of elongation at 72 C. was included together with a final elongation step at 72 C. for 7 min.

In order to transfer the transgene to next generation it is important to ensure that the transgene is present in the germ cells. Therefore DNA has been extracted from sperm cells from the two boars 4905 and 4908 . The purification of DNA was accomplished using two different procedures standard purification procedure and miniprep purification procedure 

300 L of semen was washed in 1 mL 0.9 NaCl followed by centrifugation for 5 min at 3000 rpm where after the supernatant was discarded. This step was repeated twice and 20 L Pronase 20 mg mL 20 L 1 M DTT and 300 L buffer S was added to each sample where after these were left to incubate at room temperature overnight. Subsequently 180 L 6 M NaCl was added to each sample and shaken vigorously for approximately 20 seconds. The samples were now centrifuged for 15 min. at 10000 rpm and the supernatant was then carefully transferred to a new eppendorf tube where the DNA was precipitated adding twice the volume of supernatant and centrifuged at 10000 rpm for 10 min. Subsequently the ethanol was removed and the DNA was air dried and resuspended in 300 L of nuclease free water.

300 L of semen was washed in 1 mL 0.9 NaCl followed by centrifugation for 5 min at 3000 rpm where after the supernatant was discarded. This step was repeated twice and 20 L Pronase 20 mg mL 20 L 1 M DTT and 300 L buffer S was added to each sample where after these were left to incubate at room temperature overnight. In order to enrich each sample regarding low molecular DNA and obtain a more pure DNA product the miniprep procedure from Qiagen was applied. The DNA was eluted in 200 L of nuclease free water.

DNA from both procedures was employed in the same PCR test as already mentioned and the result is shown in .

Extracts from Veterinary Declarations dated 9th of Feb. 2006 and 9th of Mar. 2006 respectively concerning transgenic animals are disclosed below.

 The boar 4905 has a bent back and very straight hocks. It raises and lays down with difficulty and stands with uneasiness in the back portion seems slightly ataxic. A significant deterioration has occurred as compared to last month. No direct signs of soreness in the limbs.

It is unclear if the cause is in the big joints at the back and pelvis region or the nerve system. Neurological symptoms of ataxia in the back portion of pigs are normally assumed to be caused by damages in the bone marrow and not the brain. Ataxia can be due to a lacking propioception i.e. feeling of the positioning of the limbs.

 White 4905 Boar in normal condition and without signs of external damages. It stands on both front and rear legs but is a little insecure when having to move and if pushed. This is clear from slight ataxic movements Crosses front legs and have cocked angles from time to time and stands with rear legs widely to the side. Strong itching reflex can be released by touching the backside. Eats and drinks normally. No signs of limping.

White 4908 Boar in normal condition and without signs of external damages. It stands with underpositioned understillede rear legs. It is slightly ataxic when turning and pushes from the side. No signs of limping.

White 4907 4911 4910 3 sows move freely. They show some signs of slight ataxic movements which can be provoked when pushing them around. 

Blood samples were withdrawn 3 days after birth and when the piglets reached 115 days blood samples were withdrawn every third week in 6 mL EDTA tubes and 6 mL serum tubes. Two sows were sacrificed at the age of 5 month and before any phenotypic changes could be observed. One 1 boar 4908 was sacrificed at the age of 14 month and another boar 4905 was sacrificed at the age of 15 month. Both boars had severe phenotypic changes. For all sacrificed animals various tissues have been snap frozen in liquid nitrogen and subsequently stored at 80 C. Furthermore various tissues including porcine brains have been fixed in formalin.

RNA was prepared from snap frozen tissues from heart kidney liver lung spleen medulla spinalis M. spinalis frontal cortex FCO parietal cortex PCO musculus longissimus dorsi M. L. dorsi musculus semitendinosus left side M.semit. l. musculus semitendinosus right side M.semit. r. musculus semibranosus left side M.semb. l. and musculus semibranosus. All DNA and RNA samples were extracted in special clean laboratory facilities under highly stringent experimental conditions using standard protocols.

50 ng of genomic DNA isolated from blood samples from each of the seven pigs were amplified using the following primers SOD1 Exon3F 5 GCTGTACCAGTGCAGGTCCTC 3 SEQ ID NO 70 and SOD1 Exon4R 5 CCATTGTGCGGCCAATGATG 3 SEQ ID NO 71 yielding a fragment of approximately 800 bp when amplifying the endogenous genomic SOD1 and approximately 200 bp when amplifying the exon 3 to exon 4 cDNA fragment. The following sample mix was employed 1 L 10 MgCl2 free reaction buffer 0.4 L 50 mM MgCl2 10 pmol of each primer 5 mM dNTP mix and 0.5 U Dynazyme Ext DNA polymerase. The reaction was performed in a total volume of 10 L and accomplished as a touchdown PCR in a GeneAmp PCR system 9700 Applied Biosystems under the following conditions Initial denaturation at 95 C. for 3 min denaturation at 95 C. for 30 sec touchdown from 62 C. to 57 C. with a decrement of 0.5 C. for sec followed by 1 min of elongation at 72 C. pr cycle. Furthermore 35 cycles of 30 sec denaturation at 95 C. 20 sec of annealing at 57 C. and 1 min of elongation at 72 C. was included together with a final elongation step at 72 C. for 7 min.

Of the total RNA 2 g was reverse transcribed from the various tissues employing a SuperScript III kit Invitrogen USA according to manufactures recommendation using random hexamer primers. RT PCR experiments were conducted in triplicate. The risk of amplifying genomic DNA was apart from primer designed to span exon exon junctions ruled out by running the PCR prior to reverse transcription. Quantitative real time PCR was performed using the TaqMan assay and PCR amplification in an ABI PE prism 7900 sequence detection system PE Applied Biosystems . Primers ssSOD1 EX4F 5 GGATCAAGAGAGGCACGTTGG 3 SEQ ID NO 72 and ssSOD1 EX4R 5 GGCGATCACAGAATCTTCGATG 3 SEQ ID NO 73 and MGB probes were designed using the Primer Express Software 2.0 PE Applied Biosystems and the MGB probes designed to match the endogenous and mutated porcine SOD1 were designed with VIC and FAM as reporter dyes SS SOD1 WT VIC 5 CAAAGATGGTGTGGCCAC 3 SEQ ID NO 74 and SS SOD1 Mut FAM 5 CAAAGATCGTGTGGCCAC 3 SEQ ID NO 75 . Furthermore the 18S ribosomal RNA gene was chosen as the endogenous control using the following primers and probes 18S F 5 CGCTCCACCAACTAAGAACG 3 SEQ ID NO 76 18S R 5 CTCAACACGGGAAACCTCAC 3 SEQ ID NO 77 and 18S probe SYBR 5 GGTGGTGG 3 SEQ ID NO 78 . Separate mixtures for mutated SOD1 wild type SOD1 and 18S were prepared and consisted of 5 L 2 TaqMan Universal PCR Master Mix 0.3 L of each primer 10 M 0.25 L probe 5 M 2 L of a 10 fold diluted cDNA template or in the case of 18s 2 L of a 10 000 fold diluted cDNA template and HO to a final volume of 10 L. Real time PCR was accomplished under the following conditions 2 min at 50 C. 10 min at 95 C. 40 cycles of 95 C. for 15 sec and 60 C. for 1 min. All PCRs were performed in triplicate. The cycle threshold Ct values corresponding to the PCR cycle number at which fluorescence emission in real time reaches a threshold above baseline emission were determined in the software SDS 2.2 PE Applied Biosystems . To compare expression levels of wild type SOD1 in the various tissues relative mRNA template concentrations were calculated using the standard curve method.

Transgene integration was determined by Southern blot analysis of DNA from musculus longissimus dorsi from the two affected boars. 15 g of genomic DNA undigested Pvu II digested and double digested with Pvu II and Bam HI were separated on a 0.9 agarose gel blotted to a nylon membrane and probed with 32P labelled SOD1 cDNA fragment constituting approximately 450 bp spanning the entire porcine SOD1 coding region derived from PCR amplification followed by nick translation. Genomic DNA from a healthy wild type pig was subjected to the same treatment as DNA from the two boars and has been included as control. Hybridization was accomplished in a hybridization buffer containing 5 Denhardt s solution and 6 SSC at 68 C. for 16 hours.

The SOD activity was determined with the Superoxide Dismutase assay kit Cayman Chemicals Ann Arbor Mich. based on the ability of SOD1 to inhibit the reduction of tetrazolium salt induced by xanthine xanthine oxidase as described 47 and was accomplished according to manufactures recommendations using serum from the two diseased pigs and one control. One unit of SOD is defined as the amount of enzyme needed to exhibit 50 dismutation of the superoxide radical O2. and a solution of bovine erythrocyte SOD Cu Zn was used as standard. Serum samples have been extracted on regular basis for 9 month yielding a total 19 samples from boar 4905 17 from 4908 and 19 from the control pig which all were included in the assay. The absorbance was read at 450 nm using a plate reader. The serum protein content was determined with a standard protein assay based on bicinchoninic acid BCA using manufactures recommendations Pierce Rockford Ill. employing bovine serum albumin as a reference and measuring the absorbance at 562 nm.

For examination of muscle tissue samples were taken from the center of longissimus dorsi above the curvature of the last rib from the two affected boars 4905 and 4908 and two wild type controls 147 and 3713 . The size of the sample was app. 5 5 1 cm. At excision the samples were frozen in isopentane cooled in liquid nitrogen to minimize internal freeze artefacts. Transverse serial sections 10 m were cut in a cryostat at 20 C. and collected on lysine coated cover slips. The sections were immunohistochemically stained for slow myosin heavy chain MHC isoform Catalogue no. CRL 2043 American Type Culture Collection to identify type I fibres. A description of the methods used for the immunohistochemical stainings is given in Pedersen et al. 2001 48 . The architecture of fibre types in pig muscles is unique as type I and IIA fibres are located in clusters. For the analysis we have counted the number of fibres within clusters as this is less affected by growth or differences in size of animals and compared the distributional characteristics of fibres within clusters. In the analysis clusters of type I fibres have been included.

For histopathological investigation of medulla spinalis both affected boars 4905 and 4908 as well as two wild type boars 147 and 3713 at approximately the same age were employed. After immersion fixation of cervical medulla spinalis in 4 paraformaldehyde the tissues were embedded in paraffin was and sectioned into slices of 5 m. Sections were stained with 7 anti SOD1 peptide antibodies. However only the 100 115 anti SOD1 peptide antibody were specific for the porcine SOD1 and is therefore the one used in this work. The immunohistochemistry was performed using the Ventana immunohistochemistry system.

Variation in number of type I muscle fibers in musculus longissimus dorsi between the two affected boars 4905 and 4908 and controls 3713 and 147 were tested with respect to statistical significance employing students t test considering unequal variances 0.05 since data approaches the underlying assumption of normality.

Initially the first signs of phenotypic alterations were appearing in one of the boars 4905 at the age of seven months. The boar was slightly ataxic and showed reduced propioception and it preferred to lie down. The symptoms became gradually worse and at the age of ten months the boar showed fasciculation when getting up and laying down. Furthermore the boar had pronounced abnormal itching reflexes and at the age of 15 months the boar was sacrificed since it was nearly unable to get up without help. The other boar 4908 got slightly symptomatic at the age of approximately 8 months where it like the other boar was slightly ataxic and showed reduced propioception. One month later it showed fasciculations which turned gradually worse and at the age of 12 month these fasciculations was very severe and continuously present when the boar was standing in an upright position. Furthermore this boar showed abnormal tongue movements. At the age of 14 months the boar was sacrificed. Both of the boars were all the time able to accomplish basic necessities of life such as eating drinking and urinating without any help.

To establish whether integration of the transgene had occurred Southern blot analysis was performed on tissue from musculus logissimus dorsi from the two affected boars and on wild type control. The used probe spans the entire SOD1 coding region constituting approximately 450 bp. However no detectable bands can be seen in any of the samples constituting undigested Pvu II digested and Pvu II and BAM HI double digested DNA from boar 4905 and 4908 and the wild type pig showing that genomic integration has not occurred. Furthermore under the same conditions the five copy fragment control harboring the SOD1 cDNA were clearly detected lane 11 and 12 in suggesting that the SOD1 cDNA is markedly underrepresented in the genome 

To further establish if the transgene could be present for instance as an extrachromosomal entity which has been demonstrated in a recent study 49 both the two affected boars and the five littermates were checked for the presence of transgene using DNA purified from blood as a template. Primers enabling both the amplification of the wild type genomic SOD1 fragment spanning exon 3 to exon 4 yielding approximately 800 bp and the mutated SOD1 DNA fragment yielding approximately 200 bp were employed. shows that apart from the positive control only the 800 bp DNA fragment arising from the endogenous SOD1 was amplified. This indicates that none of the seven animals harbour the fragment used during the SMGT procedure. Furthermore also DNA extracted from the tails of the seven animals has been subjected to PCR analysis yielding also negative results regarding the occurrence of the mutated SOD1.

Still it can not be ruled out that a minor fraction of the cells harbours the construct applied in the SMGT procedure possibly stored as an extrachromosomal fragment however this is beyond our detection limit. Furthermore the presence of transgene in other tissues in a minor fraction of the cells ruled out either however since PCR amplification in various tissues of the sacrificed pigs did not give rise to any consistent amplification of the DNA fragment used in the SMGT procedure only a minor fraction of the total number of cells include said fragment.

Expression of the SOD1 construct harbouring the Gly93Arg substitution was accomplished by quantitative RT PCR using TaqMan probes spanning the DNA region harbouring the substitution and hence separate RT PCR s were conducted regarding the wild type SOD1 mRNA and the mutagenised SOD1 mRNA from heart kidney liver lung spleen medulla spinalis frontal cortex parietal cortex musculus longissimus dorsi musculus semitendinosus left side musculus semitendinosus right side musculus semibranosus left side and musculus semibranosus from the two affected boars and from the two wild type controls and assayed for the presence of the mutagenised SOD1 transcript or alternatively altered endogenous SOD1 levels. Furthermore the mutagenised SOD1 fragment was included in the expression analysis to ensure that the SOD1 probe matching the mutagenised fragment was perfectly suited to detect any transcript. shows amplification of the mutagenised fragment in various concentrations emphasizing that the probe is suitable for detection of mutagenised transcripts. However since no difference is seen between the amplification curves representing of the affected boars and the controls in any of the 12 tissues it is concluded that mutagenised transcripts are not present in the affected boars . Although a wide variety of tissues have been included it can not be ruled out that expression of the transgene could possibly be present in other tissues or at very low levels which could not be detected in this assay.

SOD1 activity was detected spectrophotometrically in serum as a measurement for the SOD1 to inhibit the reduction of tetrazolium salt induced by xanthine xanthine oxidase 47 and was not corrected for Mn SOD meaning that the activities in represent the total SOD activity in serum. However since Mn SOD accounts for a minor fraction of the total SOD activity in serum 51 it is not likely that it would mask possible differences between affected boars and the control. The SOD activity levels in serum of the two transgenic affected boars was approximately at the same level as the control boar during the time course recorded and no significant variation across time from October 2005 August 2006 has been revealed .

Furthermore proteome analysis employing protein extracts from liver and musculus longissimus dorsi failed to detect any mutagenised SOD1 protein in the two transgenic boars data not shown .

Since the nature of muscle fibers may be used to assess disease progression of ALS the clusters including type I muscle fibers in musculus logissimus dorsi were examined regarding the number of fibers in each cluster. This investigation revealed that the frequencies of type I fibers in the two affected boars are significantly decreased in comparison to controls . Statistical evaluation reveals P values of 3 10and 2 10for 4905 and 4908 with respect to the control boar 147 emphasizing highly significant differences. Furthermore a P value of 0.08 reveals that the two controls are not statistically different which is also the case for the two affected boars P 0.11. The stainings further highlights the large differences in number of type I fibers both regarding the size of the clusters and the absolute number which is also reduced in the two transgenic boars.

Sections from cervical spinal cord of the two the two affected boars and two age matched wild type boars were stained with various peptide antibodies raised against human SOD1. However only the 100 115 anti SOD1 peptide antibody proved to be specific with respect to porcine SOD1. shows a motorneuron from the spinal cord of both an affected boar 4908 and the age matched control boar 147 . In showing a motor neuron from the affected boar SOD1 stainings are seen in the motor axon most likely arising from SOD1 inclusions. In comparison no stainings are seen in the axon of the control. Furthermore in large fluorescent compartments are seen in the neuropil which are not present in the control either.

The full length porcine synuclein SNCA cDNA was isolated from cerebellum by a combination of RT PCR and RACE. Initially blast searches using the human SNCA cDNA sequence were carried out with GenBank http www.ncbi.nlm.nih.gov Genbank index.html and with the porcine EST data bank at The Danish Institute of Agricultural Sciences DIAS . Sequence similarity search was performed with gapped alignment using NCBI Blastall with options blastn minimum value 10 8. The porcine cDNAs thus identified were used to derive oligonucleotide primers for cloning and as queries for further searches in the local genomic DIAS sequence database.

The pig cerebellum tissue used for RT PCR cloning of porcine SNCA was obtained from an adult pig. After removal tissue was dissected and pulverized in liquid nitrogen. Total RNA was isolated by the RNeasy method Qiagen . The integrity of the RNA samples was verified by ethidium bromide staining of the ribosomal RNA on 1 agarose gels.

The porcine SNCA cDNA presented here was isolated using an RT PCR cloning approach. Synthesis of cDNA was conducted with 5 g of total RNA isolated from pig cerebellum using SuperScript II RNase H reverse transcriptase Invitrogen . The cDNA synthesis was initiated by heating of total RNA oligo dT 12 18 primer dNTP at 65 C. for 5 min followed by addition of 200 units reverse transcriptase and then incubation at 42 C. for 50 min followed by 70 C. for 15 min.

The RT PCR reaction mix contained 2.5 l cDNA 1.5 mM MgCl2 0.2 mM dNTP 0.5 M of each primer SNCA F 5 CCATGGATGTATTCATGAAAGGACTTTCAA 3 SEQ ID NO 79 and SNCA R 5 CTTCCGGCTCATAGTCCTGATACCC 3 SEQ ID NO 80 and 1 U Phusion DNA polymerase Finnzymes in a total volume of 25 l. PCR amplification was carried out in the total volume using the following program Denaturation at 94 C. for 2 min. 10 cycles of 94 C. for 15 s 55 C. for 30 s 72 C. for 40 s followed by 25 cycles of 94 C. for 15 s 60 C. for 30 s 72 C. for 40 s. The PCR program was concluded by an extension at 72 C. for 7 min. Twentyfive microlitres of the amplification product was applied to a 1 agarose gel and visualized after electrophoresis by ethidium bromide staining. A fluorescent band of approx. 400 bp was cut out and eluted using the Qiaquick Gel Extraction kit from Qiagen. The eluted PCR product was cloned directly into the pCR TOPO 2.1 vector Invitrogen and sequenced in both directions.

To obtain a full length coding SNCA cDNA sense and antisense primers derived from the isolated SNCA fragment were used in 5 and 3 RACE rapid amplification of cDNA ends experiments. A sense SNCA specific primer was used in combination with a kit anchor primer Roche Molecular Biochemicals . In brief for 3 RACE an oligodT reverse transcription oligonucleotide primer 5 GACCACGCGTATCGATGTCGACTTTTTTTTTTTTTTTTV 3 V A C or G SEQ ID NO 81 was used in a reverse transcription reaction. The resultant SNCA cDNA was used as a template for PCR amplification employing the proof reading DNA polymerase Phusion Finnzymes in combination with a kit PCR anchor primer 5 GAAAACGCGTATCGATGTTCGAC 3 SEQ ID NO 82 and a gene specific primer SNCA F 5 CCATGGATGTATTCATGAAAGGACTTTCAA 3 SEQ ID NO 83 . PCR products were recovered as described above cloned into the pCR TOPO2.1 vector and sequenced. one PCR amplicon of approx. 800 bp contained a DNA fragment that showed homology to SNCA and which also matched the sequence of the partial SNCA cDNA where the sequences overlapped. For 5 RACE a reverse transcription oligonucleotide primer SNCA5 it1 5 GGATCCTACATAGAGCACACCCTC 3 SEQ ID NO 84 was used in a reverse transcription reaction. The synthesized SNCA cDNA was used as a template for PCR amplification employing Phusion DNA polymerase Finnzymes in combination with a kit PCR anchor primer 5 GAAAACGCGTATCGATGTTCGAC 3 SEQ ID NO 85 and a gene specific primer SNCA5 it2 5 TCCCGCTGCTTCTGCCACACCCTG 3 SEQ ID NO 86 . PCR products were recovered as described above cloned into the pCR TOPO2.1 vector and sequenced. one PCR product contained a DNA fragment that showed homology to SNCA and also matched the sequence of the partial SNCA cDNA where the sequences overlapped.

The interconnectedness between the original cDNA clone and the 5 and 3 RACE sequences was confirmed by PCR with the primers SNCA5 F 5 CAGTCTGTTAGGGGGAGGAGCTTATTTC 3 SEQ ID NO 87 and SNCA it3 5 CTATAGTTAATATTTATAGGTGCATAGTTCC 3 SEQ ID NO 88 . PCR amplification was carried out in the total volume using the following program Denaturation at 95 C. for 2 min. 10 cycles of touchdown 0.5 C. per cycle 95 C. for 20 s 60 C. for 30 s 72 C. for 45 s followed by 25 cycles of 95 C. for 20 s 55 C. for 30 s 72 C. for 45 s. The PCR program was concluded by an extension at 72 C. for 5 min.

DNA sequencing was performed employing the dideoxy chain termination method using BigDye terminator cycle sequencing kit with AmpliTaq DNA polymerase FS PE Applied Biosystems . The sequencing analysis was performed on an automated DNA sequencer ABI PRISM Genetic Analyzer Model 3730xl PE Applied Biosystems . Traces were aligned and visualized using the SEQUENCHER version 4.0.5 for Windows Gene Codes Corporation .

The SNCA cDNA was amplified by the reverse transcriptase polymerase chain reaction RT PCR using oligonucleotide primers derived from in silico sequences. Partial porcine SNCA EST sequences identified in the DIAS EST library were used to derive primers for RT PCR amplification of the SNCA cDNA. A 420 bp fragment was obtained which showed a high level of homology to published SNCA sequences. Sense and antisense primers were designed from this sequence and used for 5 and 3 RACE experiments. With one round of PCR a SNCA cDNA covering the coding and 3 UTR sequence was obtained. Similarly one round of PCR generated the missing 5 UTR sequence. To confirm that the obtained porcine SNCA fragments were interconnected a PCR reaction using primers covering the near full length coding sequence of was performed. This resulted in the 982 bp cDNA sequence shown in .

The identity of the porcine SNCA cDNA was established by comparing the deduced polypeptide sequence with other isolated alpha synuclein proteins. The porcine SNCA cDNA GenBank Access. No. DQ073395 is 982 bp in length with the translational start site at nucleotide 104 and the TAA stop codon at nucleotide 524. The open reading frame ORF of SsSNCA has a G C content of 50 and it encodes a polypeptide of 140 amino acids with an estimated molecular mass of 14.5 kDa and a l of 4.6. The encoded porcine synuclein protein contains the characteristic motifs of other synuclein proteins five imperfect repeats with a core consensus sequence KTKEGV distributed from the amino terminus to the central part of the protein. As for other synucleins eleven central hydrophobic amino acids are missing compared to the homologous synuclein and synuclein.

Amino acid sequence similarity between porcine synuclein and other published mammalian synuclein proteins was determined by the Clustal method. Multiple alignment of synuclein amino acids sequences from pig human mouse and rat cow chicken and revealed a very high overall homology. Interestingly it was found that the amino acid in position 53 which is an alanine in human is a threonine residue in all other species. This particular alanine residue is found substituted to a threonine Ala53Thr in familial Parkinsonism.

A very high degree of identity between porcine synuclein and most other alpha synuclein proteins was found. The encoded pig synuclein polypeptide exhibits significant sequence identity to human synuclein 98 cow synuclein 96 and mouse and rat synuclein both 94 . The lowest amino acid identity between pig synuclein and other synuclein proteins was observed with the chicken synuclein 84 and the sequence 80 . See for a phylogenetic tree. The Prosite Web site http www.expasy.ch prosite was used to analyze the 140 amino acid porcine synuclein protein sequence and predicted a molecular weight of 14.5 kDa and identified potential post translational modifications. These include one casein kinase II phosphorylation site one tyrosine kinase phosphorylation site and five putative myristoylation sites.

In order to introduce the Ala30Pro mutation into the porcine SCNA site directed mutagenesis was performed employing the QuickChange XL Site Directed Mutagenesis Kit Stratagene and accomplished in accordance with manufactures recommendation applying the following primers 

The PCR conditions were Denaturation at 95 C. for 1 min. 18 cycles of 95 C. for 50 s 60 C. for 50 s and 68 C. for 6 min. The PCR program was concluded by an extension at 68 C. for 7 min.

To ensure that the mutation of interest was integrated in the porcine SNCA gene several colonies were picked and grown overnight and plasmids were harvested and sequenced according to standard procedures. A plasmid containing the mutation was chosen as template for the subsequent procedures. Next the plasmid containing the mutated SNCA cDNA was digested with KpnI and EcoRV releasing the SNCA fragment which was now cloned into the KpnI and SmaI digested phCMV1 expression vector using standard protocols. XL10 Gold Ultracompetent Cells Invitrogen CA were transformed with the phCMV1 vector preparation and to ensure that the SNCA insert had integrated correctly into the phCMV1 vector colonies were grown in liquid LB Amp and plasmids were purified and sequenced according to standard procedures.

Vector constructs containing correctly integrated mutagenized SNCA fragments were selected for following procedures.

In order to create DNA for incubation of sperm cells large scale PCR reactions were performed. The PCR reactions were carried out in a GeneAmp PCR System 9700 Applied Biosystems in a final volume of 25 L consisting of 5 L 5 Phusion HF buffer 2 L dNTP 2.5 mM each 0.63 L forward and reverse primer 5 pmol 0.1 L Phusion DNA Polymerase 2 U L 1 L SNCA phCMV1 template and 15.6 L HO. The PCR reaction consisted of an initial denaturation at 98 C. for 30 sec followed by 30 cycles of denaturation for 10 sec at 98 C. annealing at 74 C. for 30 sec and elongation for 95 sec at 72 C. followed by a final elongation step at 72 C. for 7 min. The following primers were used to amplify the mutagenized SOD1 construct plus the flanking CMV promoter intron sequence and SV polyA generating a fragment of approximately 2100 bp.

In order to purify the generated PCR product a High Pure PCR Product Purification Kit Roche was applied. The suppliers instructions were followed throughout the purification procedure. The PCR purified fragments were sequenced to check for errors in the sequence. See .

In order to wash the porcine sperm cells and hence remove the sperm liquid the following optimized buffer is applied 

The components are dissolved in water and the solution is sterilized through a filter. Before the buffer is added to the sperm cells 6 mg ml BSA Bovine Serum Albumine Fraction V Sigma is added.

The selection of the sperm donor boars are crucial for the outcome of the procedure. A boar station Hatting KS Viborg have therefore been contacted and the boars of choice are selected so that the initial sperm motility is 90 . The sperm is collected in sterile 10 mL tubes and transported undiluted at a temperature not below 15 C. as this will cause damage to the sperm cells.

In order to choose the correct donor cells the sperm cells from the different boars are examined under a light microscope. The sperm cells originating from the boar having the highest sperm cell motility after the washing procedure are chosen. Next the sperm cells from the boar of choice are counted.

Transfer the tube to room temperature and transport it still at room temperature to stable facilities. This takes approx. 10 min. The incubated sperm cells are now ready to be applied in artificial insemination.

Two recipient sows Danish Landrace Yorkshire at approximately 140 kg were selected due to their natural heating period and used for artificial insemination 1 10DNA treated sperm sow meeting standard insemination procedures. Insemination was accomplished in the local stable areas at DIAS. Semen was collected from trained Landrace boars that have abstained for 2 days. Semen was treated according to aforementioned procedures. Both sows were examined for pregnancy 24 and 42 days after insemination showing that only one of the sows was pregnant. Animal care and experimental procedures met local national and European Union Guidelines.

After 115 days 17 Jun. 2005 10 normal looking piglets were born 5 of these were boar piglets and 5 were sow piglets. One of the boar piglets died the following day 17 Jun. 2005 . Blood samples were collected from the piglets in 6 mL EDTA blood collection tubes. Furthermore blood from a wild type animal was collected as well and handled together with the 9 aforementioned animals. At a later stage semen was collected from selected animals and genomic DNA was isolated. DNA was purified according to standard blood purification procedures in special clean laboratories in order to avoid any possible contamination.

The PCR reaction was performed in a total volume of 10 L consisting of 1 L 10 MgCl2 free reaction buffer 0.4 L 50 mM MgCl2 1 L of both forward and reverse primer 10 pmol each 0.5 L dNTP mix 0.5 L DyNazyme EXT DNA polymerase 1 U L 0.5 L DNA template 50 ng and 5.1 L HO.

The touchdown PCR reaction was accomplished in a GeneAmp PCR System 9700 Applied Biosystems under the following conditions Initial denaturation at 95 C. for 3 min denaturation at 95 C. for 30 sec touchdown from 62 C. to 57 C. with a decrement of 0.5 C. for 20 sec followed by 1 min of elongation at 72 C. pr cycle. Furthermore 35 cycles of 30 sec denaturation at 95 C. 20 sec of annealing at 57 C. and 1 min of elongation at 72 C. was included together with a final elongation step at 72 C. for 7 min.

Blood samples were withdrawn form potentially transgenic piglets and PCR reactions were carried out on purified DNA as described above. shows that all animals 4363 4371 are positive regarding the transgenic DNA fragment of 800 bp. However mosaicsm can not be ruled neither the possibility of the various animals having different copy numbers.

In order to transfer the transgene to next generation it is important to ensure that the transgene is present in the germ cells. Therefore DNA has been extracted from sperm cells from the two boars 4905 and 4908 . The purification of DNA was accomplished using a standard purification procedure 

300 L of semen was washed in 1 mL 0.9 NaCl followed by centrifugation for 5 min at 3000 rpm where after the supernatant was discarded. This step was repeated twice and 20 L Pronase 20 mg mL 20 L1 M DTT and 300 L buffer S was added to each sample where after these were left to incubate at room temperature overnight. Subsequently 180 L 6 M NaCl was added to each sample and shaken vigorously for approximately 20 seconds. The samples were now centrifuged for 15 min. at 10000 rpm and the supernatant was then carefully transferred to a new Eppendorf tube where the DNA was precipitated adding twice the volume of supernatant and centrifuged at 10000 rpm for 10 min. Subsequently the ethanol was removed and the DNA was air dried and resuspended in 300 L of nuclease free water.

Presence of the modified SNCA transgene was examined in DNA purified from sperm cells from boars 4363 4371. As shown in all boars harbored the transgene in the sperm cells. A PCR of the expected size of 800 bp can be observed in for all animals although at differential amounts.

First symptoms were observed in three boars at the age of 17 months. The pigs were examined by Knud Larsen and veterinarian Keld Dahl Winter Danish Meat . One boar 4363 had the most pronounced symptoms. When standing the boar showed a strongly upward curved convex back. Loss of muscle tissue was observed in musculus longissimus dorsi left side while the right side was unaffected. During activity permanent tremor of the tail was observed. More pronounced tremor was seen in the neck especially when the head was raised. Tremor was also visible in the tail and in ear tips. The tremor was intermittent and disappeared partly when the boar was at rest. The tremor in head and neck was worsened when the boar was surprised by visitors and by rising of the head. The boar appeared with a rigid body posture moved very slowly and did not turn around when people were circling around it. This is not a normal behaviour. The coordination of the limbs was fully normal. However the movements were very slow. The general state of health was largely unaffected and the boar did not show any signs of pain.

One month later boar 4363 was examined. Tremor of head neck and tail had increased significantly and were now present partly as resting tremor. The pronounced tremor symptoms have not been observed in any earlier described syndromes in pigs pers. comm. Keld Dahl Winter . The boar s movements were very slow and considered inhibitory as no turning towards people approaching at the tail was seen. A normal healthy pig would immediately turn towards an arriving person. Since the latest examination one month earlier the muscular atrophy had increased and spread to both sides of the back. The backline had become more visible and the neural spines were more protruding.

Boar 4363 was sacrificed and dissected. Samples from different organs such as heart liver kidney lung spleen were collected together with samples from testis and selected muscles musculus longissimus dorsi . Also the brain was taken out and divides into to halves both were dissected into discrete regions and either snap frozen or fixed in formaldehyde. After three weeks fixation different brain samples were embedded in paraffin sliced with a microtome and subjected to different stainings. Nissl AMG staining of thin layer sections from substantia nigra revealed presence of neurons but most of the neurons were shrunken with lacunae and no clear segregation between the nucleus nucleolus and cytoplasm . Similarly as shown in HE staining also demonstrated shrunken neurons with numerous lacunae holes which is a clear indication of neural degeneration.

A staining procedure for tyrosine hydrolase TH was also carried out on the thin layer sections. The results showed as illustrated in that 1 The number of dopaminergic cells seemed to be reduced and 2 The remaining dopaminergic cells and neuropil seemed to be more rough and unordered.

For comparison a brain sample from a mini pig was also examined for TH staining. Numerous dopaminergic cells were observed.

A GFAB staining revealed an intense staining in the mesencephalon of boar 4363 an indication of pronounced inflammation and gliosis . Numerous astrocytes are noted indicative of active inflammation and reactive gliosis. Very interestingly a specific antibody staining for synuclein using Ab raised against human synuclein revealed patches of large synuclein aggregates throughout the mesencephalon . synuclein s located in the cell bodies and extracellular surroundings

In conclusion all pathological examinations of boar 4363 are clearly indicative for PD. However no classical Lewy bodies are observed. Lewy bodies are a cardinal symptom of PD and presence of Lb is needed to recapitulate the development and progression of PD. The explanation for the missing LBs could easily be that boar 4363 was sacrificed too early in the progression of disease. As the boar was at the age of 18 months and showing a large row of the clinical symptoms of PD but without complete resting tremor it was what could be expected at this particular stage.

Pig brain lymphocyte and liver RNA was isolated with the TRI reagent Sigma . For RT PCR of PSEN1 the following primers were used PSEN1forward 5 TGGAGGAGAACACATGAAAGAAAG 3 SEQ ID NO 95 PSEN1 forward EcoR1 5 GGGGAATTCTGGAGGAGAACACATGAAAGAAAG 3 SEQ ID NO 96 PSEN1reverseEcoR1 5 GGGGAATTCCCTGACTTTGTTAGATGTGGACAC 3 SEQ ID NO 97 . The RT PCR reaction was incubated at 50 C. for 60 min with the reverse primer followed by PCR with the PSEN1forward EcoR1 and PSEN1 reverse EcoR1 primers at conditions 94 C. for 3 min 35 cycles of 94 C. 45 sec 62 C. 30 sec 68 C. 2 min followed by a final elongation step at 68 C. for 7 min . Amplified DNA fragments were purified from agarose gels and either directly sequenced or EcoR1 cloned into pCDNA3 followed by DNA purification and sequencing. For RT PCR of PSEN2 the following primers were used PSEN2 forward 5 GCCATGCTCACTTTCATGGC 3 PSEN2 reverse SEQ ID NO 98 5 CACGACTGCGTCCAGTGACC 3 SEQ ID NO 99 . The reverse transcription reaction was accomplished using the Invitrogen reverse transcription system Invitrogen and 5 g of total RNA according to the manufacturer s instructions. Subsequently the PCR reaction was carried out at the following conditions 94 C. for 3 min 35 cycles of 94 C. 45 sec 60 C. 30 sec 68 C. 2 min followed by a final elongation step at 68 C. for 7 min . Amplified DNA fragments were purified from agarose gels and either directly sequenced or cloned into pCR 2.1 TOPO Vector Invitrogen followed by DNA purification and sequencing. The porcine pSEN1 and pSEN2 cDNA sequences were submitted to GenBank Accession numbers DQ853416 and DQ853415 respectively 

Radioactive probes were generated employing the nick translation kit from Invitrogen which incorporated 32P dCTP into the PCR generated PSEN1 exon8 fragment. High density colony BAC filters a generous gift from Dr. P. D. Jong of the porcine genome were screened with the PSEN1 exon8 probe. The filters were pre hybridized hybridized washed and autoradiographed according to standard methods. Positive spots were localised and BAC DNA of positive clones was isolated using the alkaline lysis method described by Zhang et al. 1996 . BAC clone 388G9 contained the PSEN1 genomic sequence and was used for intronic sequence generation.

The BAC clone 388G9 was sequenced with primers located in exons 5 7 8 and 9 and pointing towards the intronic sequences. Table 8 shows the applied primers. All exon and flanking intronic sequences were deposited as a gapped submission to GenBank Accession number DQ86246 .

Exons 5 7 8 and 9 and flanking intron sequences were amplified by PCR primers listed in table 8 under SNP screening application . Exon 5 and flanking intron sequences were amplified at conditions 50 ng DNA 94 C. for 3 min and 35 cycles 94 C. 30 sec 60 C. 20 sec 72 C. 1 min. Exon 7 and flanking intron sequences were amplified at conditions 50 ng DNA 94 C. for 3 min and 35 cycles 94 C. 20 sec 58 C. 20 sec 72 C. 1 min. Exon 8 and flanking intron sequences were amplified at conditions 50 ng DNA 94 C. for 3 min and 35 cycles 94 C. 45 sec 64 C. 30 sec 72 C. 1 min. Exon 9 and the flanking intron sequences were amplified at conditions 50 ng DNA 94 C. for 3 min and 35 cycles 94 C. 20 sec 58 C. 20 sec 72 C. 1 min. All PCR products were incubated with exozap at 37 C. for 1 hour and sequenced with the forward amplification primer. The sequences were analyzed using PolyBace and checked manually in Consed.

A porcine rodent somatic cell hybrid panel was used for physical mapping Yerle et al. 1996 of both PSEN1 and PSEN2. For PSEN1 the exon 9 forward and reverse primers 2 were used for amplification of the probe fragment. For PSEN2 the PCR primers PSEN2exon12F 5 GTTTGTGTCTGACCCTCCTGCTGC 3 SEQ ID NO 125 and PSEN2exon12R 5 CAGATGTAGAGCTGGTGGGGAGG 3 SEQ ID NO 126 were used for amplification of the probe fragment. PCR s were performed in a total volume of 10 L containing 10 ng DNA 1 PCR buffer 2.5 mM of each dNTP 5 pmol of each primer and 0.5 U of Taq polymerase Bioline under the following conditions 94 C. for 3 min 35 cycles of 94 C. for 20 s 65 C. for 20 s and 72 C. for 20 s and a final elongation step for 5 min at 72 C.

Fetal pig brains were immersion fixed in formalin and paraffin embedded tissue blocks were produced from various brain regions. 10 micrometer coronal sections were then obtained on coated glass slides. The sections were deparaffinized and pretreated with proteinase K for 6 min. The slides were blocked with BSA 1 mg ml for 10 min. Immunohistochemical demonstration of PSEN1 and PSEN2 was performed using the EnVision System HRP DAB DAKO . The anti PSEN1 antibody was a rabbit polyclonal antiserum 520 a generous gift from Dr. Poul Fraser Toronto Canada used in 1 100 dilution with incubation time 2 hours. The anti PSEN2 antibody was the mouse monoclonal antibody APS 26 used in 1 33 dilution with incubation time 2 hours abcam . Nuclei were counterstained in haematoxylen solution. The slides were finally coverslipped with Faramount Aqueous Mounting Medium DAKO .

Total RNA was isolated from cerebellum frontal cortex hippocampus brainstem and basal ganglia from 60 80 100 and 115 days old porcine fetuses using the TRI Reagent Sigma in compliance with the manufacturer s instructions. Three separate tissues were applied for each type of tissue and time in gestation yielding a total of 60 samples. The reverse transcription reaction was accomplished using an Invitrogen reverse transcription system Invitrogen and 5 g of RNA according to the manufacturer s instructions. Quantitative real time PCR was performed using the TaqMan assay and PCR amplification in an ABI PE prism 7900 sequence detection system PE Applied Biosystems . Primers and MGB probes were designed using the Primer Express Software 2.0 PE Applied Biosystems so that both forward and reverse primer spanned an exon exon junction. The MGB probe was synthesized with VIC as a reporter dye. After an initial screening with different control genes GAPDH was chosen as the endogenous control and the MGB probe was synthesized with VIC as a reporter dye. The primers and probes are detailed in table 8. Separate mixtures for PSEN1 PSEN2 and GAPDH were prepared and consisted of 5 L 2 TaqMan Universal PCR Master Mix 0.3 L of each primer 10 M 0.25 L probe 5 M 2 L of a 5 fold diluted cDNA template and HO to a final volume of 10 L. Real time PCR was done under the following conditions 2 min at 50 C. 10 min at 95 C. 40 cycles of 95 C. for 15 sec and 60 C. for 1 min. For both PSEN1 PSEN2 and GAPDH PCRs were performed in triplicate. The cycle threshold Ct values corresponding to the PCR cycle number at which fluorescence emission in real time reaches a threshold above baseline emission were determined in SDS 2.2 PE Applied Biosystems . To compare expression patterns in the various brain tissues at different developmental stages mRNA template concentrations for GAPDH PSEN1 and PSEN2 were calculated using the standard curve method. Standard curves were constructed using 8 fold dilution of day 115 frontal cortex cDNA 4 2 1 0.5 0.25 0.125 and 0.0625 L . The mRNA quantity of each amplicon was calculated for each standard and experimental sample.

The equality of PSEN1 and PSEN2 expression levels between different time of gestation within the 5 sampled tissues were tested for statistical significance using the standalone software REST 52 . The statistical model applied was the Pair Wise Fixed Reallocation Randomisation Test. The assumption regarding normal distribution of the data was avoided and differences in expression between groups were assessed using the means for statistical significance by randomization. The level of probability was set at P

To determine the cDNA sequence of porcine PSEN1 we designed a set of primers based on the conserved 5 and 3 untranslated regions between the rodent bovine and human PSEN1. Using RT PCR the cDNA representing the entire pig PSEN1 open reading frame was amplified cloned and sequenced. The porcine cDNA was throughout the sequence homologous to human PSEN1 90 but only homologous to human PSEN2 in short dispersed regions. The porcine PSEN1 protein has a length of 467 amino acids which compares well with the 467 amino acids of the human and mouse counterparts . Multiple amino acid sequence alignment of PSEN1 revealed a 92 sequence identity between pig and human . Furthermore 34 changes were observed between the two sequences 11 of these being conservative. Comparison of pig and mouse revealed a sequence identity of 89 and 16 of 50 amino acid changes were conservative. The cow PSEN1 has a length of 478 amino acids and a sequence identity of 94 to porcine 12 of 28 changes being conservative and hence cow PSEN1 is the PSEN1 variant that shows the highest degree of identity to the porcine counterpart. Mutations in human PSEN1 can be cause Alzheimer s disease and it is noteworthy that none of the amino acid changes between pig and human are located in positions known to cause Alzheimer s disease. www.molgen.ua.ac.be ADMutations . By contrast at position 318 where human PSEN1 contains a non pathogenic polymorphism E318G a Q residue is present at the equivalent position in pig cow and mouse PSEN1 53 . Two other non pathogenic polymorphisms R35Q and F175S present in human PSEN1 are in the porcine PSEN1 R and F respectively 54 55 .

The human PSEN2 cDNA sequence was used to blast NCBI porcine databases as well as an in house porcine database allowing the design of primers corresponding to the 5 end of the coding region and the 3 non coding region of the putative porcine PSEN2. The primers amplified a porcine cDNA fragment of approximately 1.4 kb and sequence analysis revealed high sequence homology with human PSEN2 92 but only homologous to human PSEN1 cDNA in small dispersed regions demonstrating that the porcine orthologous of PSEN2 was identified. Analysis of the porcine PSEN2 open reading frame showed that the porcine PSEN2 protein consisted of 448 amino acid residues as do the human and mouse orthologous . Multiple amino acid sequence alignment of PSEN2 revealed 97.8 sequence identity between pig and human and 1 of the observed 10 changes was conservative . Comparison of pig and mouse showed a sequence identity of 95 8 of the 20 changes being conservative. The cow PSEN2 has a length of 449 amino acids and a sequence identity of 98.2 where 4 of the 8 changes are of conservative. Thus as observed for PSEN1 also the bovine PSEN2 protein shows the highest degree of identity to the porcine counterpart. Moreover it should be noted that none of the observed changes between the sequences of the different species are located in PSEN2 positions identified to be mutated in Alzheimer s disease patients www.molgen.ua.ac.be ADMutations . At position 334 a non pathogenic polymorphism P334R has been identified in human PSEN2 54 56 and the proline residue was conserved in the porcine cDNA.

The amino acid sequence for porcine PSEN1 shows 64 identity to porcine PSEN2 and particularly amino acids in the transmembrane domains and the C terminus are conserved data not shown . Also the two aspartic acid residues located in transmembrane domain 6 D257 in PSEN1 and D263 in PSEN2 and transmembrane domain 7 D385 in PSEN1 and D363 in PSEN2 and the PAL sequence P433 A434 L435 in PSEN1 and P414 A415 L416 in PSEN2 are conserved in both porcine PSEN1 and PSEN2 consistent with the essential role of these residues for the protease catalytic function of the presenilins 54 57 58 .

A porcine rodent somatic cell hybrid panel was used for the chromosomal mapping of porcine PSEN1 and PSEN2 genes data not shown 59 . Statistical evaluation applying the Interpreting PCR data program http www.toulouse.inra.fr lgc pig pcr pcr.htm showed a chromosomal localization for the PSEN1 gene to chromosome 7q12 q26 with a probability of 0.4494 and a correlation of 1 and for the PSEN2 gene to chromosome 10p11 p16 with a probability of 0.9959 and a correlation of 0.7255. The specified regions of porcine chromosomes 7 and 10 have synteny with human chromosomes 14 and 1 respectively. This is in agreement with mapping of the human PSEN1 gene to 14q24.3 and the human PSEN2 gene to 1q31 q42.

To examine for genetic variation in porcine PSEN1 we resequenced exons 5 7 8 and 9 in a large animal material consisting of 900 Landrace Yorkshire crossbreed sows and a pig breed panel consisting of 55 Landrace Duroc Yorkshire and Hampshire breeds. These exons were chosen for sequence analysis because they constitute hotspots for mutations in familiar Alzheimer s disease. However no SNP s were identified in the 4 exonic regions data not shown . Next we extended the polymorphism analysis to include intronic sequences which identified a C T SNP at position 58 in intron 8 position 1163 in the sequence deposited in DQ86246 as well as two C T polymorphisms at positions 52 and 92 and a G A polymorphism at position 117 in intron 10 positions 1535 1575 and 1600 in the DQ86246 . The genotyping data are summarized in table 9 and 10. All breeds except Hampshire were polymorphic in intron 8 and at positions 52 and 92 in intron 10 whereas only the Yorkshire breed was polymorphic at position 117 in intron 10. Genotype frequencies were in accordance with Hardy Weinberg equilibrium indicating that no selective disadvantage is associated with the SNPs.

PSEN1 and PSEN2 have been shown to be widely expressed during embryonic development and especially the expression profile in the CNS is well characterized 60 62 . Here we measured the mRNA expression levels of PSEN1 and PSEN2 in hippocampus cerebellum frontal cortex basal ganglia and brain stem from dissected porcine foetus brains at days 60 80 100 and 115 of gestation using three biological samples for each of the time points. Day E115 corresponds to the normal day of birth. The PCR analyses were performed in triplicates. The requirement for a proper internal control gene was met by normalization to the GAPDH expression level to compensate for inter PCR variation with respect to RNA integrity and sample loading. Although housekeeping gene expression has been reported to vary considerably within different tissues or treatments we did not find any significant differential expression of GAPDH within the 5 different porcine brain tissues at the various developmental stages. The standard curve for the control GAPDH R 0.98 PSEN1 R 0.98 and PSEN2 R 0.98 were generated by plotting Ct values versus log L of cDNA. The slope of the regression line was used to calculate the amount of cDNA and thus mRNA in each sample. All GAPDH cDNA s generated almost identical Ct values within each type of tissue data not shown and accordingly the mRNA expression levels of PSEN1 and PSEN2 were normalized to the GAPDH expression level. Ethidium bromide staining after real time PCR confirmed specific amplification of the relevant PCR products data not shown .

PSEN1 and PSEN2 were expressed in all 5 tissues at the 4 time points evaluated. However it should be noted that for both PSEN1 and PSEN2 the mean standard deviation is considerable reflecting a high heterogeneity among animals. In basal ganglia the PSEN1 expression levels did not vary significantly between the different times of gestation . In frontal cortex cerebellum and hippocampus the PSEN1 expression level was significantly lower at day 115 of gestation compared to day 60 P 0.001 P 0.036 and P 0.003 respectively yielding a reduction of 5 2 and 3 times for the said tissues . Furthermore the reduction in PSEN1 expression in frontal cortex is also significant at day 80 compared to day 60 P 0.003 . Similarly PSEN1 expression is gradually reduced in hippocampus during the time period of gestation . Moreover the same tendency is seen in cerebellum however the reduction in expression levels is only significant between day 100 and 115 P 0.015 and day 60 and 115 P 0.036 .

For PSEN2 no differential expression was observed in frontal cortex. In hippocampus the only significant variation was seen as an increase in expression level between day 60 and 80 of gestation P 0.015 . Also in the brain stem PSEN2 is upregulated between day 60 and 80 of gestation P 0.032 . In cerebellum and basal ganglia the expression levels of PSEN2 are up regulated between day 80 and 100 P 0.003 and P 0.03 . When comparing the overall expression levels of PSEN1 and PSEN2 an approximately three fold lower PSEN2 expression level is observed. In conclusion the real time PCR analysis showed significant albeit small alterations in the expression levels of PSEN1 and PSEN2 mRNA in different brain compartments during embryonic brain development which likely reflect biological importance.

To examine the localization of the PSEN1 and PSEN2 proteins in situ we utilized immunohistochemical stainings at embryonic day 100 brain slides with antibodies for PSEN1 and PSEN2. PSEN1 staining was more intense and diffusible outside cell bodies than observed for the PSEN2 staining . We note that all PSEN2 stained regions also were positive for PSEN1 staining . No clear alterations in localization or intensity of PSEN1 and PSEN2 staining were detected in analysis of other embryonic time points or brain regions data not shown . Intracellular immunostaining was confined to the cytoplasm with a distinct sparing of the nuclei. The immunostaining was observed in all parts of the CNS especially in neurons but also to some extend in astrocytes and data not shown . In cortex both pyramidal and nonpyramidal cells were stained . Also all hippocampus CA subfields and the granule cells were PSEN positive . The immunohistochemical analysis supports that the PSEN proteins are located in the majority if not all of the neurons. Moreover all PSEN2 stained cell types were also positive for PSEN1 staining in accordance with the observed redundancies in PSEN1 and PSEN2 functions 63 64 .

Human TNR disease causing regions are in most cases also identifiable in the primate and rodent genomes 65 67 . However in rodents the TNR regions in general are composed of significantly fewer TNR units and are less polymorphic. As TNR sequences are rapidly evolving and may functionally influence the affected genes changes in such regions have the potential to participate in functional diversification 65 68 . To analyse how the disease causing TNR regions identified in humans have evolved in the porcine genome we analysed porcine TNRs. We here describe that in terms of TNR tract lengths the porcine TNRs in general represent an intermediate between rodent and humans and that several of the TNRs are polymorphic in the pig. In addition the length of TNRs was in several of the porcine loci comparable to the lengths normally identified in primates.

Genomic DNA was prepared from unrelated no common parents and grandparents Duroc Landrace Hampshire Yorkshire and Goettingen minipig males according to standard procedures. The DNA was isolated from EDTA stabilized blood using a salting out procedure 69 .

To sequence porcine genomic TNR regions flanking sequences conserved between mouse and humans were identified and corresponding PCR primers designed. 50 ng of Duroc pig genomic DNA was used in PCR with conditions 95 C. 30 58 C. 30 72 C. 1 cycles. Standard taq polymerase PCR conditions were used except for the inclusion of 1 M Betaine and 5 DMSO. After agarose gel electrophoresis analysis DNA of expected size was purified and sequenced. If the intensity of bands or the sequencing result was evaluated inadequate for further analysis new PCR primers were designed either based on the evolutionary approach or nested according to determined sequences. By this scheme the following optimized primer sets were used to amplify genomic TNR regions all PCR reactions run for 35 cycles with 50 ng genomic DNA as input SCA1 SCA1 CAGCGCTCCCAGCTGGAGG SEQ ID NO 127 SCA1 GGAYGTACTGGTTCTGCTGG SEQ ID NO 128 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSO. SCA2 SCA2NyTRI GCCACCGTAGAGGAGGAGGAAG SEQ ID NO 129 SCA2TRI CTCACCATGTCGCTGAAGC SEQ ID NO 130 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSO. SCA3 SCA3 I7 CCATGGGAATAGTTTTTCTCATG SEQ ID NO 131 SCA3exon10 GGTTGGCTTTTCACATGGATGTG SEQ ID NO 132 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSO. SCA6 SCA6nyTRI CGGCCACACGTGTCCTATTC SEQ ID NO 133 SCA6NYTRI GGCCGCTGGGGGCCGCTCG SEQ ID NO 134 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSo. SCA7 SCA7Tri GGAGCGGAAAGAATGTCGGAG SEQ ID NO 135 SCA7Tri CCCACAGATTCCACGACTGTC SEQ ID NO 136 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSO. SCA17 pTBP GAAGAGCTGTGGAGTCTGG SEQ ID NO 137 pTBP CTATCCATTTTGGAGGAGCAG SEQ ID NO 138 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSo. DRPLA Drpla Ny GGAGGCCAGTCCACTGCTCAC SEQ ID NO 139 Drpla Ny GGGAGACATGGCATAAGGGTG SEQ ID NO 79 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSO. SMBA ARTri X ARTri X 94 C. 45 58 C. 30 72 C. 2 . HD pHUNNYRE CCGCCATGGCGACCCTGGAAA SEQ ID NO 140 pHUNNYRE GGTGGCGGCTGAGGAGGCTG SEQ ID NO 141 95 C. 30 65 C. 30 72 C. 1 with betaine and DMSO. FMR1 FMR1 Ny 1 CGTTTCGGTTTCACTTCCGGTG SEQ ID NO 142 FMR1Zoo CCGCACTTCCACCACCAGCTC SEQ ID NO 143 95 C. 30 60 C. 30 72 C. 1 with betaine and DMSO. FMR2 FMR2 Ny TGCGGCGGCAGCAGCCGCTAC SEQ ID NO 1444 FMR2 Ny 2 CCCCTGTGAGTGTGTAAGTGTG SEQ ID NO 145 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSo. SCA12 SCA12TRI GGGAGGAGCCTCGCCTTTAATG SEQ ID NO 146 SCA12Tri CGCGACAAAATGGTGCCTTTC SEQ ID NO 147 95 C. 30 58 C. 30 72 C. 1 with betaine and DMSO. DMPK gDMPKpoly GCCCTGCTGCCTTCTCTAGGTC SEQ ID NO 148 gDMPKpoly CCCCAGCTCTAGCCCTGTGATC SEQ ID NO 149 94 C. 30 64 C. 30 72 C. 2 . DNA fragments were purified from agarose gels using GFX columns Amersham Biosciences and sequenced according to standard procedures. DNA fragments were amplified from Duroc Landrace Hampshire Yorkshire and Goettingen minipig male genomic DNA.

Sequence information for TNRs in human Pan troglodytes chimpanzee dog rat mouse and opossum was extracted from genome browsers at NCBI www.ncbi.nlm.nih.gov Genomes .

Genotyping of different Huntingtin allele lengths was performed by microsatellite analysis in a porcine material consisting of 14 Duroc boars 611 Landrace and Landrace Yorkshire crossbred sows and 349 offspring originating from 4 of the Duroc boars and the aforementioned sows. The following primers were used Fw FAM CCGCCATGGCGACCCTGGAAA SEQ ID NO 150 Rw GGTGGCGGCTGAGGAGGCTG SEQ ID NO 151 . The amplicon was amplified in a total volume of 10 L in a mixture consisting of 50 ng of genomic DNA 5 pmol of each primer 2.5 mM dNTPs 1 L of 10 reaction buffer 0.5 L DMSO 2.5 L 4 M Betaine and 5 U Taq DNA polymerase Applied Biosystems USA . PCR amplification was as follows initial denaturation for 4 min at 95 C. and 35 cycles at 95 C. for 30 sec 64 C. for 30 sec 72 C. for 1 min. An extension step of 72 C. for 5 min was added after the final cycle. PCR products were denatured with formamide and electrophoresis was carried out on a 3730 DNA Analyzer Applied Biosystems USA using the recommended protocol. Size analyses of DNA fragments were accomplished with the GeneMapper Software Ver 3.0 Applied Biosystems USA . The internal size standard GeneScan LIZ 500 Applied Biosystems USA was employed for allele sizing.

The determined porcine TNR regions and flanking sequences were submitted to genebank and have been assigned the following accession numbers SCA1 DQ915251 DQ915252 SCA2 DQ915254 SCA3 DQ915255 DQ915256 SCA6 DQ915259 DQ915260 DQ915261 SCA7 DQ915262 SCA17 DQ915258 DRPLA DQ915263 DQ915264 SMBA DQ915257 HD DQ915274 DQ915275 DQ915276 DQ915277 DQ915278 DQ915279 DQ915280 DQ915281 DQ915282 FMR1 DQ915269 DQ915270 DQ915271 DQ915272 DQ915273 FMR2 DQ915268 SCA12 DQ915265 DQ915266 DQ915267 DMPK DQ915253 .

To sequence porcine genomic regions homologous to human disease causing TNRs we first identified flanking sequences showing highly conserved regions between mouse and humans. Employing this PCR approach 12 porcine genomic loci corresponding to human disease causing TNRs were amplified. The identified TNRs were located in coding regions and 5 UTRs. The DMPK 3 UTR TNR was amplified using primers based on EST sequences available in Genbank. To search for TNR polymorphisms in different porcine breeds we included Duroc Landrace Hampshire Yorkshire and Goettingen minipig. For each of the pig breeds a number of animals were analysed assuring the detection of common alleles allele frequency 10 .

Porcine genomic sequences homologous to human non coding TNR expansion regions. We first addressed porcine genomic sequences homologous to human non coding TNRs.

DMPK In the 3 UTR of the human myotonic dystrophy protein kinase gene DMPK a CTG TNR is located 19 23 . The normal size of this TNR varies between 5 and 37. Expansions from above 50 to several thousand CTG repeats result in myotonic dystrophy. A CTG TNR consisting of 4 CTG repeats in all the pig breeds studied Duroc Landrace Yorkshire Hampshire and minipigs was identified at the same localization in the porcine DMPK. No length variation in the DMPK TNR was identified . The repeat number found is below the minimum number of CTG repeats 5 CTGs observed in humans. In the mouse the TNR sequence of DMPK is composed of 2 CAG repeats flanked by single CTG repeats . In dog and rat a single CTG is present flanked by other types of TNRs .

SCA12 The SCA12 CAG TNR within the 5 UTR of the human PPP2R2B gene normally varies in size from 7 to 28 repeats and in the expanded form from above 65 to 78 TNRs 27 . Three alleles consisting of 8 9 and 10 CAG repeats were identified in the porcine breeds . The most abundant allele in Duroc contained 8 CAG repeats whereas the most abundant allele in Hampshire contained 10 CAG repeats . An allele containing 9 CAG repeats was the only allele observed in Landrace Yorkshire and Minipigs . The presence of long uninterrupted CAG TNRs in the porcine SCA12 locus is distinct from the mouse and rat SCA12 locus which is composed of CAG TNRs interrupted with CAC and GAG triplets . Notably the lengths of the porcine SCA12 alleles are comparable and even higher than SCA12 allele lengths identified in some humans .

FMR1 FRAXA In the promoter region of the human FMR1 gene 6 to 52 CGG repeats are normally present 24 25 . Expansions in the range of 55 to 200 repeats result in the pre mutation while the full mutation ranges from 200 to several thousand repeats resulting in fragile X syndrome. The sequence of the FMR1 TNR region in Duroc showed the presence of two alleles a 14 CGG repeat allele with a frequency of 90 and a 13 CGG repeat allele with a frequency of 10 . Sequence analysis of the FMR1 TNR in the other porcine breeds showed a high degree of FMR1 TNR length polymorphisms. A 15 CGG repeat allele was identified in Duroc Hampshire and Yorkshire . This allele was the most common in Yorkshire whereas a 9 CAG repeat allele was also identified in Hampshire and a 12 CAG repeat allele in Duroc and Minipigs. No allele polymorphism was observed in Landrace. All the porcine FMR1 alleles were longer than the homologous mouse TNR sequence which is composed of 6 CGG repeats and 2 CGG repeats separated by a CGA . The dog FMR1TNR has a size similar to the pig TNR . In the most common type of human FMR1 TNR allele two groups of CGG repeats are separated by an AGG triplet . Also the chimpanzee FMR1TNR is highly polymorphic and includes AGG triplet interruptions 70 . Interestingly the porcine FMR1TNR length exceeds the minimal length present in the human FMR1CGG TNR prone to expand.

FMR2 FRAXE The number of CCG repeats in the TNR of the 5 end of the human FMR2 gene varies from 6 to 35 26 . Expansions containing from 61 to 200 repeats result in the pre mutation and expansions above 200 repeats result in the full mutation and the fragile X syndrome. The homologous region in the porcine FMR2 gene was found to contain 7 CCG repeats in all breeds analysed . The porcine TNR length exceeds the length of the homologous mouse TNR which is composed of 4 CCG repeats . Furthermore the length of the porcine CCG TNR is longer than the minimal CCG allele length identified in humans .

Next we addressed the sequence of poly glutamine coding TNR sequences of nine porcine loci. SCA1 In the human SCA1 locus CAG TNR expansions in the ATX1 protein causes spinocerebellar ataxia 71 73 . The human SCA1 TNR region is characterized by the presence of 12 CAG repeats followed by two CAT repeats flanking a CAG triplet 28 . The CAG TNR prone to expand is normally composed of between 6 and 39 repeats and the expanded version consists of 41 to 81 repeats. The porcine SCA1 TNR is composed of two CAG repeats separated by eight proline encoding triplets . However in Minipigs a variant was detected the most common allele having a CAG TNR duplication . The dog SCA1 region includes 6 CAG repeats . The mouse and rat homologous region is composed of two CAG repeats and three proline coding triplets . Thus in terms of CAG repeat numbers the porcine and rodent SCA1 TNRs are similar but distinct from the homologous human TNR . However due to the presence of numerous proline codons CCX the porcine SCA1 TNR have increased complexity compared to the TNR in rodents.

SCA2 The SCA2 TNR expansion affecting the ATX2 protein results in spinocerebellar ataxia 73 . This TNR normally consists of 15 to 30 CAG repeats and the expanded form ranges from 35 to 59 triplets 29 . The porcine locus is composed of 7 CAG repeats separated by two CAA triplets . Accordingly the region encodes a stretch of nine poly glutamines. No polymorphism whatsoever was observed in the porcine SCA2 TNR region. The dog SCA2 TNR had proline interruptions in the poly glutamine stretch . The homologous rodent SCA2 TNR is composed of CAG repeats separated by a proline encoding triplet .

SCA3 In the human SCA3 locus a CAG TNR expansion in the ataxin 3 gene above 54 repeats results in ataxia whereas the normal number of CAG repeats varies between 12 and 36 30 32 . In pigs a five CAG TNR allele was identified in Duroc Hampshire and Landrace whereas five and six CAG TNR alleles were identified in Yorkshire and Minipigs . In terms of the number of encoded poly glutamines the porcine SCA3 TNR region is homologous to the mouse SCA3 TNR region and well below the critical number in the human counterpart. The dog SCA3 TNR encodes 12 glutamines but includes several CAA interrupting triplets .

SCA6 The SCA6 TNR expansion in the CACNA1A voltage dependent calcium channel results in ataxia 33 . The normal number of TNRs is between 4 and 18 and expansions from 21 to 27 TNRs are disease causative. In pigs a SCA6 allele was identified composed of 5 and 4 CAG repeats separated by a CAA triplet thereby encoding a poly glutamine stretch of 10 . In Minipigs longer SCA6 alleles composed of 7 or 9 CAG repeats followed by the CAA triplet and 4 CAG repeats were identified. These alleles encode stretches of 12 and 14 poly glutamines respectively. The poly glutamine stretches encoded by the porcine SCA6 TNR region were comparable in length to the normal range encoded by the human SCA6 sequence and the 14 poly glutamine stretch identified in Minipigs matches the upper range of the more common human alleles . In dog a 10 CAG SCA6 TNR was present . We note the absence of a SCA6 TNR in rodents and a high degree of divergence in the CACNA1A sequence between rodents and pig dog and primates at the particular genomic position.

SCA7 The TNR of the human SCA7 locus in the N terminal end of the ataxin 7 protein is normally composed of 7 to 35 CAG repeats 54 . Disease causing expansions range from 37 to 200 repeats. The TNR of the porcine SCA7 locus contains 5 CAG repeats and no polymorphisms were observed in the breeds . Similarly the TNR of the mouse SCA7 locus contains 5 CAG repeats . Interestingly a SCA7 allele with 5 CAG repeats has also been identified in humans. Note that the SCA7 CAG repeats are flanked by a poly alanine stretch and a glutamine and proline rich stretch highly variable between the examined mammalian genomes data not shown .

DRPLA CAG expansions within the human atrophin 1 gene results in dentatorubral pallidoluysian atrophy DRPLA 36 . The normal range of repetitive CAG repeats is from 3 to 25 and in patients with DRPLA allele sizes have expanded to 49 to 88 CAG repeats. The most common natural occurring human allele encodes a stretch of 17 poly glutamines. In the porcine atrophin 1 TNR six CAG repeats flanked by multiple CAG and CAA triplets resulting in an allele encoding 14 poly glutamines was identified. Moreover in Minipigs an allele with seven CAGs resulting in a 15 poly glutamine encoding allele was observed . This means that the length of the porcine atrophin 1 poly glutamine stretches is above the minimal length observed in humans. The TNR of the mouse atrophin 1 gene encodes six glutamines with an interrupting proline and from the rat gene is encoded 11 glutamines highly interrupted by proline residues . The dog atrophin 1 TNR encodes a stretch of 12 poly glutamines . In mammalians the atrophin 1 TNR is flanked by histidine rich stretches polymorphic between the examined mammalian genomes data not shown .

SCA17 A CAG expansion in the TATA box binding protein TBP gene is causative of the SCA17 phenotype resulting in ataxia 35 . The human TNR region is composed of two groups of CAG repeats separated by multiple CAA and CAG triplets. Expansions normally progress from the larger of the two CAG groups. The normal stretch of encoded poly glutamines varies between 29 and 42 whereas poly glutamine stretches from 47 to 55 have been identified in SCA17 patients. The porcine SCA17 TNR region encodes 26 poly glutamines and thus is the largest poly glutamine encoding TNR identified in pigs . The pig SCA17 sequence is composed of four groups of CAG TNRs intervened by CAA triplets . The longest CAG group consists of 10 CAGs. No allele polymorphism was identified in porcine SCA17. In comparison mouse SCA17 TNR encodes 13 poly glutamines and a maximum of three CAG triplets in one stretch . The dog SCA17 TNR encodes 22 glutamines but includes several alanine interruptions .

SBMA CAG repeat expansions in exon 1 of the androgen receptor AR gene on the X chromosome results in spinal and bulbar muscular atrophy Kennedy s disease 37 . The normal length of the human CAG TNR is between 11 and 33 CAG copies and in diseased individuals the expansion ranges from 38 to 62. In the pig an AR allele encoding 7 poly glutamines interrupted by a single CTG leucine triplet was identified . No TNR variation was observed between the different porcine breeds. In mouse and rat the AR TNR sequence encodes 3 glutamines interrupted by a single AGG arginine triplet or CGG arginine triplet respectively . The dog AR TNR is composed of a 10 CAG repeats . Note in mammalians the TNR is flanked by a proline and glutamine rich stretch highly polymorphic between the examined mammalian genomes data not shown .

HD The CAG TNR in the Huntingtin gene is located in the 5 end of the coding region. Normally the gene contains from 6 to 35 CAG repeats and in Huntington s disease patients more than 35 CAG repeats are present 74 . A large degree of variation was observed in the pig Huntingtin TNR region . This difference was due both to a variable number of CAG repeats but also due to the absence or presence of a CAA triplet which separates the continuous CAG theme into two groups . Alleles with sizes encoding 13 to 24 poly glutamines from were identified. Highly interesting in Duroc an allele composed of a stretch of 21 uninterrupted CAGs a CAA triplet and two CAGs resulting in the encoding of a total of 24 poly glutamines was identified. This allele represents the largest number of uninterrupted CAGs identified in the analysis of porcine TNRs. The long allele was specifically identified in Duroc and has an allele frequency of 20 . The Minipig Huntingtin gene and polymorphisms therein were described previously 75 . The numbers of poly glutamines encoded by porcine Huntingtin TNRs are indeed comparable to the human repeat and are for the 24 poly glutamine allele even above the number of poly glutamines most frequently found in human Huntingtin alleles. In contrast the mouse Huntingtin TNR region encodes seven poly glutamines intervened by a CAA triplet and the dog TNR region 10 poly glutamines . The Huntingtin TNR is flanked by a proline and glutamine rich stretch polymorphic between the examined mammalian genomes data not shown .

Since a long uninterrupted CAG TNR sequence was present in the Huntingtin gene of Duroc pigs we next examined if this sequence was stably inherited or prone to retractions or expansions. For this purpose we used a porcine material consisting of Duroc boars crossed with Landrace and Landrace Yorkshire crossbread sows. The genotyping of different Huntingtin allele lengths was performed by microsatellite analysis. From the boar cohort four heterozygous boars were identified having both a 161 bp fragment and a 140 bp fragment corresponding to 24 and 17 glutamines respectively. The genotyping result of the 4 heterozygous boars was verified by DNA sequencing. The genotyping data from the cohort of the 611 sows used in the breeding scheme resulted in the identification of the alleles also present in the pure Landrace and Yorkshire breeds. Furthermore two new Huntingtin alleles encoding 14 and 15 glutamines not present in the pure breeds were identified in the sow cohort table 11 . Also the 24 poly glutamine encoding allele was present in the sow population however only at a low frequency 0.3 table 11 . Interestingly no alleles were identified with a size less than 13 poly glutamines or a polyglutamine number of 19 20 21 22 or 23 table 1 and data not shown . However since the population of sows was not completely unrelated the allele frequency calculations are only indicative. The group of genotyped offspring consisted of 349 pigs. All these 349 animals had a genotype in accordance with the inheritance of a maternal and paternal Huntingtin allele without any TNR retractions or expansions table 12 . Thus we could not identify any evidence for transmission instability of Huntingtin TNRs.

The paternal genotype was 140 161. The sow haplotypes are indicated together with the frequency F within the breeding cohort. For all offspring a genotype could be assigned in accordance with transmission of both paternal and maternal alleles not subjected to expansions or retractions. A total of 349 offspring were genotyped. The genotypes are visualized according to the length L of the analysed fragments. Q indicates the number of glutamines encoded from the TNR alleles. N indicates the number of offspring with the indicated genotype.

Only a very limited amount of knowledge is available regarding the presence of collagen X in permanent cartilages such as trachea in comparison to growth plate cartilages. We isolated the full length cDNA encoding collagen X from porcine trachea illustrating that collagen X is not solely present in hypotrophic chondrocytes of calcifying matrix typically present in long bones. However in humans collagen X have previously been shown in trachea especially in elderly individuals where ossification occur in the tracheal cartilage as a result of the progressing age. Furthermore in developing rat tracheal cartilage collagen X was confined to the peripheral uncalcified region of the cartilage 76 indicating that collagen X might play a role beside providing the molecular structural environment in relation to endochondral ossification however this is not confirmed in humans 77 .

The pig trachea tissue used for RT PCR cloning of COLA was obtained from an adult pig. Tissue was dissected and pulverized in liquid nitrogen after removal. Total RNA was isolated by RNeasy method Qiagen . The integrity of RNA samples was verified by ethidium bromide staining of the ribosomal RNA on 1.5 agarose gels.

Generation of a porcine COL10A1 clone was accomplished in the following way RNA derived from adult porcine trachea was employed in a cDNA synthesis using conditions where 5 g of total RNA was mixed with 1 L of oligo dT 12 18 500 g mL and DEPC treated HO to a final volume of 12 L. The mixture was incubated at 70 C. for 10 min after which 4 L of 5 first strand buffer 2 L of 0.1 mM DTT 1 L of 10 mM dNTP mix and 1 L 200 U L of Superscript II Invitrogen was added and the sample was further incubated at 42 C. for 1 hour followed by an inactivation step at 70 C. for 15 min. oligonucleotides used for RT PCR cloning were derived from the genomic COL10A1 sequence Accession number AF222861 and also contained linkers Bgl II in the sense primer and Eco RI in the antisense primer for subsequent cloning. The RT PCR reaction mix contained 2.5 L cDNA 1.5 mM MgCl2 0.2 mM dNTP 10 pmol of each primer COL F 5 AACAGATCTATGCTGCCACAAACAGCCCTTTTGCT 3 SEQ ID NO 152 and COL R 5 GCAGAATTCTCACATTGGAGCCACTAGGAATCCT 3 SEQ ID NO 153 and 1.0 U Phusion Fidelity DNA polymerase Finnzymes in a total volume of 25 L using the following conditions denaturation at 98 C. for 2 min. followed by 30 cycles of 98 C. for 10 s. 60 C. for 30 s. and 72 C. for 1 min. The PCR program was concluded by a final extension step at 72 C. for 5 min. The PCR was accomplished in a GeneAmp PCR System 9700 Applied Biosystems . The amplification product was applied to a 1 ethidium bromide stained agarose gel and a fluorescent band of approximately 2000 bp was isolated using standard procedures and cloned into the pCR 2.1 TOPO vector Invitrogen CA and sequenced in both forward and reverse direction applying standard procedures to ensure that they harboured the COL10A1 amplicon. The COL10A1 plasmid DNA was digested with Eco RI and ligated into a phCMV1 Gene Therapy Systems Inc. expression vector pre digested with Eco RI. The successful cloning of COL10A1 into the phCMV1 vector was confirmed by sequencing.

In order to create DNA for incubation of sperm cells large scale PCR reactions were performed. The PCR reactions were carried out in a GeneAmp PCR System 9700 Applied Biosystems in a final volume of 25 L consisting of 5 L 5 Phusion HF buffer 2 L dNTP 2.5 mM each 0.63 L forward and reverse primer 5 pmol 0.1 L Phusion DNA Polymerase 2 U L 1 L COL10A1 CMV1 template and 15.6 L HO. The PCR reaction consisted of an initial denaturation at 98 C. for 30 sec followed by 30 cycles of denaturation for 10 sec at 98 C. annealing at 74 C. for 30 sec and elongation for 95 sec at 72 C. followed by a final elongation step at 72 C. for 7 min. The following primers were used to amplify the COL10A1 construct plus the flanking CMV promoter intron enhancer sequence and SVpolyA generating a fragment of approximately 3600 bp.

In order to purify the generated PCR product a High Pure PCR Product Purification Kit Roche was applied. The suppliers instructions were followed throughout the purification procedure. The PCR purified fragments were sequenced to check for errors in the sequence.

First and second semen ejaculate were collected from 8 different boars yielding 16 semen fractions in total. All fractions of spermatozoa had an initial motility of 90 prior to the washing procedure. Seminal fluid was quickly removed by washing the sperm in Fertilization Buffer FB consisting of 56.1 g Glucose 3.5 g EDTA 2HO 3.5 g Sodium Citrate 2HO and 1.1 g sodium bicarbonate dissolved in 1 liter of sterilized water. Furthermore 6 mg ml BSA Fraction V Sigma was added. Briefly 5 mL of FB BSA prewarmed to 37 C. was added to 5 mL of undiluted semen and left for 5 minutes at room temperature. Next FB BSA at room temperature was added to 50 ml and centrifuged for 10 minutes at room temperature at 500 g. The supernatant was removed and semen was resuspended in 50 mL FB BSA at room temperature and further centrifuged at 500 g at 17 C. after which the supernatant was removed again and the spermatozoa was resuspended in 15 mL of FB BSA. Next in order to select the optimal donor cells the spermatozoa from the different boars were quickly examined under a light microscope. The sperm cells originating from the boar having the highest sperm cell motility after the washing procedure were chosen as vehicles for the subsequent transgenic procedures. Furthermore the atozoa were counted.

1 10sperm cells from the chosen donor boar were incubated for 100 minutes at 17 C. with the linear COL10A1 DNA fragment in a concentration of 0.4 g DNA 10spermatozoa in a suspension of 120 mL FB BSA. The container was inverted every 20 minutes to prevent sedimentation of spermatozoa. Finally the mixture was incubated 10 minutes at room temperature and employed in artificial insemination of a sow in natural heat.

Semen was collected from trained Danish Landrace boars that had abstained for 2 days. one recipient sow Danish Landrace Yorkshire at approximately 140 kg were selected due to its natural heating period and used for artificial insemination 1 10DNA treated spermatozoa sow meeting standard insemination procedures. Insemination was accomplished in the local stable areas at DIAS. The sow was examined for pregnancy 24 and 42 days after insemination showing that was successfully pregnant. After ended gestation period 6 boar and 6 sow piglets were naturally born and blood samples were withdrawn from the piglets three days after birth in EDTA and serum tubes. Due to economical reasons 9 animals were sacrificed and hence 2 boars and 1 sow piglet were kept for later investigation. The sow was sacrificed at the age of 7 month since severe phenotypic alterations were present. Animal care and experimental procedures met local national and European Union Guidelines.

DNA was prepared from EDTA stabilized blood samples from the 12 piglets. RNA was prepared from snap frozen tissues from the sacrificed sow heart kidney liver lung skin ovary musculus longissimus dorsi musculus semimembranosus and musculus triceps brachii . To avoid any contamination all DNA and RNA samples were extracted in special clean laboratory facilities under highly stringent experimental conditions using standard protocols.

To ensure the presence of the transgene 50 ng of genomic DNA from each animal isolated from blood samples were amplified using the following primers phCMV430F 5 GTCTCCACCCCATTGACGTC 3 SEQ ID NO 156 and phCMV646R 5 GGATCGGTCCCGGTGTCTTC 3 SEQ ID NO 157 yielding a fragment of 217 bp using the following sample mix 1 L 10 MgClfree reaction buffer 0.4 L 50 mM MgCl 10 pmol of each primer 5 mM dNTP mix and 0.5 U Dynazyme Ext DNA polymerase. The reaction was performed in a total volume of 10 L and accomplished as a touchdown PCR in a GeneAmp PCR system 9700 Applied Biosystems under the following conditions Initial denaturation at 95 C. for 3 min denaturation at 95 C. for 30 sec touchdown from 62 C. to 57 C. with a decrement of 0.5 C. for 20 sec followed by 1 min of elongation at 72 C. pr cycle. Furthermore 35 cycles of 30 sec denaturation at 95 C. 20 sec of annealing at 57 C. and 1 min of elongation at 72 C. was included together with a final elongation step at 72 C. for 7 min.

Transgene integration was determined by Southern blot analysis of DNA from musculus longissimus dorsi from the affected sow. 10 g of genomic DNA from the affected sow and from a wild type pig was digested with Bgl II and double digested with Bgl II and Bam HI respectively and separated on a 0.9 agarose gel blotted to a nylon membrane and probed with 32P labelled collagen X NC1 fragment derived from PCR amplification using the following primers NC1 F 5 GCTCTAGAGGTCCCACCCACCCGAAGG 3 SEQ ID NO 158 and NC1 R 5 TCTCTAGATCACATTGGAGCCACTACGAA 3 SEQ ID NO 159 .

From the right foreleg tissues from the growth plates physis and metaphyses of femur ulna and radius were sampled together with the articular areas including the articular epiphyseal cartilage complex of the shoulder elbow and carpal joints. From the two hind legs similar tissue samples were collected from the hip and knee joints i.e. from the femoral and tibial bones. Also the osteo chondral junction of three ribs was sampled for histology. All tissues were fixed in 10 neutral buffered formalin followed by decalcification in a solution containing 3.3 formaldehyde and 17 formic acid for 2 weeks. The tissues were processed through graded concentrations of alcohol and xylene and embedded in paraffin wax blocks. Tissue sections of 4 5 were stained by haematoxylin and eosin HE and in selected cases by Van Gieson for collagen and safranin o for cartilage matrix 78 79 .

In order to establish transgenic pigs which could shed light on the functional role of collagen X a COL10A1 cDNA wild type construct was generated. Interestingly the porcine COL10A1 cDNA was isolated from trachea using primers generated from the previously known COL10A1 sequence 80 . The cDNA fragment was initially cloned into the pCR 2.1 TOPO vector and subsequently successfully removed to the phCMV1 expression vector facilitating constitutive expression qua the CMV promoter. In order to impede possible truncation of important elements following SMGT the COL10A1 DNA fragment employed in the transgenic procedure include additional nucleotides 5 and 3 prime to the CMV promoter and the polyA fragment which constitutes a fragment of approximately 3600 bp in total .

Initially eight Danish Landrace boars were used as sperm donors and the sperm fraction showing the highest motility after the initial washing procedure introduced to remove seminal fluid was chosen in the subsequent DNA incubation procedure. After ended gestation period 12 normal looking piglets were naturally born and PCR analysis of DNA isolated from blood amplifying a 217 bp DNA fragment located in the 5 prime phCMV fragment of the construct showed that all 12 piglets harboured the transgene see . However due to economical reasons only 3 piglets were kept for phenotypical investigations.

At the age of approximately 5 month one of the three transgenic animals a sow developed clinical manifestations of lameness. The sow was oppressed in its movements rising from primarily difficulties with the forelegs had a toddling gait a curved back which however could be a compensation for the abnormalities arising from the fore legs. The signs became slightly worse and at the age of approximately seven month the sow would only walk when forced to and it would immediately bent down on the elbows and walk on these. However for ethical reasons the sow was sacrificed at this point.

Southern blot analysis shown in performed on tissue from musculus longissimus dorsi from the diseased sow and a normal wild type pig reveals additional bands in the affected pig when digested with Bgl II and Bam HI in comparison to the control using the entire collagen NC1 domain as a radiolabelled probe. This therefore led us to conclude that the transgene has become integrated into the genome of the affected pig. Unfortunately Bgl II did only digest the genomic DNA to a limited degree and hence no difference is seen regarding additional bands in the unidigestion using this enzyme alone lane 1 and 2 .

Expression of the COL10A1 construct was accomplished by RT PCR using primers located in COL10A1 exon 2 and COL10A1 exon 3 on total RNA isolated musculus triceps brachii ovary kidney skin liver lung musculus longissmus dorsi musculus semimembranosus heart and liver spleen kidney lung and heart from a wild type control. The RT PCR analysis revealed expression of collage X in kidney lung and heat in the affected sow although only very limited amounts of transcript are present in kidney . However since the PCR is only qualitative the judgement of expression levels within the samples should be taken with precaution although the same amount of total RNA has been used in the cDNA synthesis. Furthermore no expression of COL10A1 is present in the wild type control tissues.

The articular epiphyseal area of the humeral trochlea constituting the proximal portion of the elbow joint revealed severe alterations see . The cartilage was retained with contents of hypertrophied chondrocytes. Within areas of retained cartilage formation of clefts were regularly seen and in the depth irregular cavitations filled with fibrinous material were present along with eosinophilic streaks. Moreover the distal growth plates of both ulna and radius were irregular with localized areas of retained cartilage. Lesions were not present within sections from the ribs the articular epiphyseal areas or the growth plates of the hind legs.

Thus in conclusion the integration and expression of the COL10A1 transgene gave rise to a dyschondroplasia phenotype affecting especially the two fore limbs of a sow. The alterations present in the transgeneic sow resembles osteochondrosis which is known to affect younger animals and is characterised by similar lesions of the articular epiphyseal cartilage complex 41 as observed in the forelegs of the present sow.

